14:52:46.666 [Thread-48-KafkaPrintDataSpout-executor[1 1]] ERROR o.a.k.c.c.i.ConsumerCoordinator - [onComplete,768] - [Consumer clientId=consumer-1, groupId=springboot-kafka-storm] Offset commit with offsets {test_topic-0=OffsetAndMetadata{offset=14, metadata=''}} failed
org.apache.kafka.clients.consumer.CommitFailedException: Commit cannot be completed since the group has already rebalanced and assigned the partitions to another member. This means that the time between subsequent calls to poll() was longer than the configured max.poll.interval.ms, which typically implies that the poll loop is spending too much time message processing. You can address this either by increasing the session timeout or by reducing the maximum size of batches returned in poll() with max.poll.records.
	at org.apache.kafka.clients.consumer.internals.ConsumerCoordinator.sendOffsetCommitRequest(ConsumerCoordinator.java:808)
	at org.apache.kafka.clients.consumer.internals.ConsumerCoordinator.doCommitOffsetsAsync(ConsumerCoordinator.java:641)
	at org.apache.kafka.clients.consumer.internals.ConsumerCoordinator.commitOffsetsAsync(ConsumerCoordinator.java:608)
	at org.apache.kafka.clients.consumer.KafkaConsumer.commitAsync(KafkaConsumer.java:1486)
	at org.apache.kafka.clients.consumer.KafkaConsumer.commitAsync(KafkaConsumer.java:1455)
	at org.apache.kafka.clients.consumer.KafkaConsumer.commitAsync(KafkaConsumer.java:1431)
	at com.example.dmy.kafkastorm.storm.spout.KafkaPrintDataSpout.nextTuple(KafkaPrintDataSpout.java:69)
	at org.apache.storm.daemon.executor$fn__10727$fn__10742$fn__10773.invoke(executor.clj:654)
	at org.apache.storm.util$async_loop$fn__553.invoke(util.clj:484)
	at clojure.lang.AFn.run(AFn.java:22)
	at java.lang.Thread.run(Thread.java:748)
15:54:59.111 [Thread-41-PrintOutBolt-executor[2 2]] ERROR c.e.d.k.s.s.PrintOutBolt - [execute,64] - Bolt的数据处理失败!数据:["Thu Feb 21 15:32:30 CST 2019","Thu Feb 21 15:54:36 CST 2019","Thu Feb 21 15:54:38 CST 2019","Thu Feb 21 15:54:40 CST 2019","Thu Feb 21 15:54:42 CST 2019","Thu Feb 21 15:54:44 CST 2019","Thu Feb 21 15:54:46 CST 2019","Thu Feb 21 15:54:48 CST 2019","Thu Feb 21 15:54:50 CST 2019","Thu Feb 21 15:54:52 CST 2019","Thu Feb 21 15:54:54 CST 2019","Thu Feb 21 15:54:56 CST 2019","Thu Feb 21 15:54:58 CST 2019"]
java.util.NoSuchElementException: null
	at java.util.ArrayList$Itr.next(ArrayList.java:854)
	at com.example.dmy.kafkastorm.storm.bolt.PrintOutBolt.execute(PrintOutBolt.java:53)
	at org.apache.storm.daemon.executor$fn__10795$tuple_action_fn__10797.invoke(executor.clj:739)
	at org.apache.storm.daemon.executor$mk_task_receiver$fn__10716.invoke(executor.clj:468)
	at org.apache.storm.disruptor$clojure_handler$reify__10135.onEvent(disruptor.clj:41)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchToCursor(DisruptorQueue.java:509)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchWhenAvailable(DisruptorQueue.java:487)
	at org.apache.storm.disruptor$consume_batch_when_available.invoke(disruptor.clj:74)
	at org.apache.storm.daemon.executor$fn__10795$fn__10808$fn__10861.invoke(executor.clj:861)
	at org.apache.storm.util$async_loop$fn__553.invoke(util.clj:484)
	at clojure.lang.AFn.run(AFn.java:22)
	at java.lang.Thread.run(Thread.java:748)
15:55:01.102 [Thread-41-PrintOutBolt-executor[2 2]] ERROR c.e.d.k.s.s.PrintOutBolt - [execute,64] - Bolt的数据处理失败!数据:["Thu Feb 21 15:55:00 CST 2019"]
java.util.NoSuchElementException: null
	at java.util.ArrayList$Itr.next(ArrayList.java:854)
	at com.example.dmy.kafkastorm.storm.bolt.PrintOutBolt.execute(PrintOutBolt.java:53)
	at org.apache.storm.daemon.executor$fn__10795$tuple_action_fn__10797.invoke(executor.clj:739)
	at org.apache.storm.daemon.executor$mk_task_receiver$fn__10716.invoke(executor.clj:468)
	at org.apache.storm.disruptor$clojure_handler$reify__10135.onEvent(disruptor.clj:41)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchToCursor(DisruptorQueue.java:509)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchWhenAvailable(DisruptorQueue.java:487)
	at org.apache.storm.disruptor$consume_batch_when_available.invoke(disruptor.clj:74)
	at org.apache.storm.daemon.executor$fn__10795$fn__10808$fn__10861.invoke(executor.clj:861)
	at org.apache.storm.util$async_loop$fn__553.invoke(util.clj:484)
	at clojure.lang.AFn.run(AFn.java:22)
	at java.lang.Thread.run(Thread.java:748)
15:55:03.127 [Thread-41-PrintOutBolt-executor[2 2]] ERROR c.e.d.k.s.s.PrintOutBolt - [execute,64] - Bolt的数据处理失败!数据:["Thu Feb 21 15:55:02 CST 2019"]
java.util.NoSuchElementException: null
	at java.util.ArrayList$Itr.next(ArrayList.java:854)
	at com.example.dmy.kafkastorm.storm.bolt.PrintOutBolt.execute(PrintOutBolt.java:53)
	at org.apache.storm.daemon.executor$fn__10795$tuple_action_fn__10797.invoke(executor.clj:739)
	at org.apache.storm.daemon.executor$mk_task_receiver$fn__10716.invoke(executor.clj:468)
	at org.apache.storm.disruptor$clojure_handler$reify__10135.onEvent(disruptor.clj:41)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchToCursor(DisruptorQueue.java:509)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchWhenAvailable(DisruptorQueue.java:487)
	at org.apache.storm.disruptor$consume_batch_when_available.invoke(disruptor.clj:74)
	at org.apache.storm.daemon.executor$fn__10795$fn__10808$fn__10861.invoke(executor.clj:861)
	at org.apache.storm.util$async_loop$fn__553.invoke(util.clj:484)
	at clojure.lang.AFn.run(AFn.java:22)
	at java.lang.Thread.run(Thread.java:748)
15:55:05.149 [Thread-41-PrintOutBolt-executor[2 2]] ERROR c.e.d.k.s.s.PrintOutBolt - [execute,64] - Bolt的数据处理失败!数据:["Thu Feb 21 15:55:04 CST 2019"]
java.util.NoSuchElementException: null
	at java.util.ArrayList$Itr.next(ArrayList.java:854)
	at com.example.dmy.kafkastorm.storm.bolt.PrintOutBolt.execute(PrintOutBolt.java:53)
	at org.apache.storm.daemon.executor$fn__10795$tuple_action_fn__10797.invoke(executor.clj:739)
	at org.apache.storm.daemon.executor$mk_task_receiver$fn__10716.invoke(executor.clj:468)
	at org.apache.storm.disruptor$clojure_handler$reify__10135.onEvent(disruptor.clj:41)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchToCursor(DisruptorQueue.java:509)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchWhenAvailable(DisruptorQueue.java:487)
	at org.apache.storm.disruptor$consume_batch_when_available.invoke(disruptor.clj:74)
	at org.apache.storm.daemon.executor$fn__10795$fn__10808$fn__10861.invoke(executor.clj:861)
	at org.apache.storm.util$async_loop$fn__553.invoke(util.clj:484)
	at clojure.lang.AFn.run(AFn.java:22)
	at java.lang.Thread.run(Thread.java:748)
15:55:07.169 [Thread-41-PrintOutBolt-executor[2 2]] ERROR c.e.d.k.s.s.PrintOutBolt - [execute,64] - Bolt的数据处理失败!数据:["Thu Feb 21 15:55:06 CST 2019"]
java.util.NoSuchElementException: null
	at java.util.ArrayList$Itr.next(ArrayList.java:854)
	at com.example.dmy.kafkastorm.storm.bolt.PrintOutBolt.execute(PrintOutBolt.java:53)
	at org.apache.storm.daemon.executor$fn__10795$tuple_action_fn__10797.invoke(executor.clj:739)
	at org.apache.storm.daemon.executor$mk_task_receiver$fn__10716.invoke(executor.clj:468)
	at org.apache.storm.disruptor$clojure_handler$reify__10135.onEvent(disruptor.clj:41)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchToCursor(DisruptorQueue.java:509)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchWhenAvailable(DisruptorQueue.java:487)
	at org.apache.storm.disruptor$consume_batch_when_available.invoke(disruptor.clj:74)
	at org.apache.storm.daemon.executor$fn__10795$fn__10808$fn__10861.invoke(executor.clj:861)
	at org.apache.storm.util$async_loop$fn__553.invoke(util.clj:484)
	at clojure.lang.AFn.run(AFn.java:22)
	at java.lang.Thread.run(Thread.java:748)
15:55:09.196 [Thread-41-PrintOutBolt-executor[2 2]] ERROR c.e.d.k.s.s.PrintOutBolt - [execute,64] - Bolt的数据处理失败!数据:["Thu Feb 21 15:55:08 CST 2019"]
java.util.NoSuchElementException: null
	at java.util.ArrayList$Itr.next(ArrayList.java:854)
	at com.example.dmy.kafkastorm.storm.bolt.PrintOutBolt.execute(PrintOutBolt.java:53)
	at org.apache.storm.daemon.executor$fn__10795$tuple_action_fn__10797.invoke(executor.clj:739)
	at org.apache.storm.daemon.executor$mk_task_receiver$fn__10716.invoke(executor.clj:468)
	at org.apache.storm.disruptor$clojure_handler$reify__10135.onEvent(disruptor.clj:41)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchToCursor(DisruptorQueue.java:509)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchWhenAvailable(DisruptorQueue.java:487)
	at org.apache.storm.disruptor$consume_batch_when_available.invoke(disruptor.clj:74)
	at org.apache.storm.daemon.executor$fn__10795$fn__10808$fn__10861.invoke(executor.clj:861)
	at org.apache.storm.util$async_loop$fn__553.invoke(util.clj:484)
	at clojure.lang.AFn.run(AFn.java:22)
	at java.lang.Thread.run(Thread.java:748)
15:55:11.223 [Thread-41-PrintOutBolt-executor[2 2]] ERROR c.e.d.k.s.s.PrintOutBolt - [execute,64] - Bolt的数据处理失败!数据:["Thu Feb 21 15:55:10 CST 2019"]
java.util.NoSuchElementException: null
	at java.util.ArrayList$Itr.next(ArrayList.java:854)
	at com.example.dmy.kafkastorm.storm.bolt.PrintOutBolt.execute(PrintOutBolt.java:53)
	at org.apache.storm.daemon.executor$fn__10795$tuple_action_fn__10797.invoke(executor.clj:739)
	at org.apache.storm.daemon.executor$mk_task_receiver$fn__10716.invoke(executor.clj:468)
	at org.apache.storm.disruptor$clojure_handler$reify__10135.onEvent(disruptor.clj:41)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchToCursor(DisruptorQueue.java:509)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchWhenAvailable(DisruptorQueue.java:487)
	at org.apache.storm.disruptor$consume_batch_when_available.invoke(disruptor.clj:74)
	at org.apache.storm.daemon.executor$fn__10795$fn__10808$fn__10861.invoke(executor.clj:861)
	at org.apache.storm.util$async_loop$fn__553.invoke(util.clj:484)
	at clojure.lang.AFn.run(AFn.java:22)
	at java.lang.Thread.run(Thread.java:748)
15:55:13.244 [Thread-41-PrintOutBolt-executor[2 2]] ERROR c.e.d.k.s.s.PrintOutBolt - [execute,64] - Bolt的数据处理失败!数据:["Thu Feb 21 15:55:12 CST 2019"]
java.util.NoSuchElementException: null
	at java.util.ArrayList$Itr.next(ArrayList.java:854)
	at com.example.dmy.kafkastorm.storm.bolt.PrintOutBolt.execute(PrintOutBolt.java:53)
	at org.apache.storm.daemon.executor$fn__10795$tuple_action_fn__10797.invoke(executor.clj:739)
	at org.apache.storm.daemon.executor$mk_task_receiver$fn__10716.invoke(executor.clj:468)
	at org.apache.storm.disruptor$clojure_handler$reify__10135.onEvent(disruptor.clj:41)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchToCursor(DisruptorQueue.java:509)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchWhenAvailable(DisruptorQueue.java:487)
	at org.apache.storm.disruptor$consume_batch_when_available.invoke(disruptor.clj:74)
	at org.apache.storm.daemon.executor$fn__10795$fn__10808$fn__10861.invoke(executor.clj:861)
	at org.apache.storm.util$async_loop$fn__553.invoke(util.clj:484)
	at clojure.lang.AFn.run(AFn.java:22)
	at java.lang.Thread.run(Thread.java:748)
15:55:15.266 [Thread-41-PrintOutBolt-executor[2 2]] ERROR c.e.d.k.s.s.PrintOutBolt - [execute,64] - Bolt的数据处理失败!数据:["Thu Feb 21 15:55:14 CST 2019"]
java.util.NoSuchElementException: null
	at java.util.ArrayList$Itr.next(ArrayList.java:854)
	at com.example.dmy.kafkastorm.storm.bolt.PrintOutBolt.execute(PrintOutBolt.java:53)
	at org.apache.storm.daemon.executor$fn__10795$tuple_action_fn__10797.invoke(executor.clj:739)
	at org.apache.storm.daemon.executor$mk_task_receiver$fn__10716.invoke(executor.clj:468)
	at org.apache.storm.disruptor$clojure_handler$reify__10135.onEvent(disruptor.clj:41)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchToCursor(DisruptorQueue.java:509)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchWhenAvailable(DisruptorQueue.java:487)
	at org.apache.storm.disruptor$consume_batch_when_available.invoke(disruptor.clj:74)
	at org.apache.storm.daemon.executor$fn__10795$fn__10808$fn__10861.invoke(executor.clj:861)
	at org.apache.storm.util$async_loop$fn__553.invoke(util.clj:484)
	at clojure.lang.AFn.run(AFn.java:22)
	at java.lang.Thread.run(Thread.java:748)
15:55:17.288 [Thread-41-PrintOutBolt-executor[2 2]] ERROR c.e.d.k.s.s.PrintOutBolt - [execute,64] - Bolt的数据处理失败!数据:["Thu Feb 21 15:55:16 CST 2019"]
java.util.NoSuchElementException: null
	at java.util.ArrayList$Itr.next(ArrayList.java:854)
	at com.example.dmy.kafkastorm.storm.bolt.PrintOutBolt.execute(PrintOutBolt.java:53)
	at org.apache.storm.daemon.executor$fn__10795$tuple_action_fn__10797.invoke(executor.clj:739)
	at org.apache.storm.daemon.executor$mk_task_receiver$fn__10716.invoke(executor.clj:468)
	at org.apache.storm.disruptor$clojure_handler$reify__10135.onEvent(disruptor.clj:41)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchToCursor(DisruptorQueue.java:509)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchWhenAvailable(DisruptorQueue.java:487)
	at org.apache.storm.disruptor$consume_batch_when_available.invoke(disruptor.clj:74)
	at org.apache.storm.daemon.executor$fn__10795$fn__10808$fn__10861.invoke(executor.clj:861)
	at org.apache.storm.util$async_loop$fn__553.invoke(util.clj:484)
	at clojure.lang.AFn.run(AFn.java:22)
	at java.lang.Thread.run(Thread.java:748)
15:55:19.309 [Thread-41-PrintOutBolt-executor[2 2]] ERROR c.e.d.k.s.s.PrintOutBolt - [execute,64] - Bolt的数据处理失败!数据:["Thu Feb 21 15:55:18 CST 2019"]
java.util.NoSuchElementException: null
	at java.util.ArrayList$Itr.next(ArrayList.java:854)
	at com.example.dmy.kafkastorm.storm.bolt.PrintOutBolt.execute(PrintOutBolt.java:53)
	at org.apache.storm.daemon.executor$fn__10795$tuple_action_fn__10797.invoke(executor.clj:739)
	at org.apache.storm.daemon.executor$mk_task_receiver$fn__10716.invoke(executor.clj:468)
	at org.apache.storm.disruptor$clojure_handler$reify__10135.onEvent(disruptor.clj:41)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchToCursor(DisruptorQueue.java:509)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchWhenAvailable(DisruptorQueue.java:487)
	at org.apache.storm.disruptor$consume_batch_when_available.invoke(disruptor.clj:74)
	at org.apache.storm.daemon.executor$fn__10795$fn__10808$fn__10861.invoke(executor.clj:861)
	at org.apache.storm.util$async_loop$fn__553.invoke(util.clj:484)
	at clojure.lang.AFn.run(AFn.java:22)
	at java.lang.Thread.run(Thread.java:748)
15:55:21.335 [Thread-41-PrintOutBolt-executor[2 2]] ERROR c.e.d.k.s.s.PrintOutBolt - [execute,64] - Bolt的数据处理失败!数据:["Thu Feb 21 15:55:20 CST 2019"]
java.util.NoSuchElementException: null
	at java.util.ArrayList$Itr.next(ArrayList.java:854)
	at com.example.dmy.kafkastorm.storm.bolt.PrintOutBolt.execute(PrintOutBolt.java:53)
	at org.apache.storm.daemon.executor$fn__10795$tuple_action_fn__10797.invoke(executor.clj:739)
	at org.apache.storm.daemon.executor$mk_task_receiver$fn__10716.invoke(executor.clj:468)
	at org.apache.storm.disruptor$clojure_handler$reify__10135.onEvent(disruptor.clj:41)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchToCursor(DisruptorQueue.java:509)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchWhenAvailable(DisruptorQueue.java:487)
	at org.apache.storm.disruptor$consume_batch_when_available.invoke(disruptor.clj:74)
	at org.apache.storm.daemon.executor$fn__10795$fn__10808$fn__10861.invoke(executor.clj:861)
	at org.apache.storm.util$async_loop$fn__553.invoke(util.clj:484)
	at clojure.lang.AFn.run(AFn.java:22)
	at java.lang.Thread.run(Thread.java:748)
15:55:23.384 [Thread-41-PrintOutBolt-executor[2 2]] ERROR c.e.d.k.s.s.PrintOutBolt - [execute,64] - Bolt的数据处理失败!数据:["Thu Feb 21 15:55:23 CST 2019"]
java.util.NoSuchElementException: null
	at java.util.ArrayList$Itr.next(ArrayList.java:854)
	at com.example.dmy.kafkastorm.storm.bolt.PrintOutBolt.execute(PrintOutBolt.java:53)
	at org.apache.storm.daemon.executor$fn__10795$tuple_action_fn__10797.invoke(executor.clj:739)
	at org.apache.storm.daemon.executor$mk_task_receiver$fn__10716.invoke(executor.clj:468)
	at org.apache.storm.disruptor$clojure_handler$reify__10135.onEvent(disruptor.clj:41)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchToCursor(DisruptorQueue.java:509)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchWhenAvailable(DisruptorQueue.java:487)
	at org.apache.storm.disruptor$consume_batch_when_available.invoke(disruptor.clj:74)
	at org.apache.storm.daemon.executor$fn__10795$fn__10808$fn__10861.invoke(executor.clj:861)
	at org.apache.storm.util$async_loop$fn__553.invoke(util.clj:484)
	at clojure.lang.AFn.run(AFn.java:22)
	at java.lang.Thread.run(Thread.java:748)
15:55:25.409 [Thread-41-PrintOutBolt-executor[2 2]] ERROR c.e.d.k.s.s.PrintOutBolt - [execute,64] - Bolt的数据处理失败!数据:["Thu Feb 21 15:55:24 CST 2019"]
java.util.NoSuchElementException: null
	at java.util.ArrayList$Itr.next(ArrayList.java:854)
	at com.example.dmy.kafkastorm.storm.bolt.PrintOutBolt.execute(PrintOutBolt.java:53)
	at org.apache.storm.daemon.executor$fn__10795$tuple_action_fn__10797.invoke(executor.clj:739)
	at org.apache.storm.daemon.executor$mk_task_receiver$fn__10716.invoke(executor.clj:468)
	at org.apache.storm.disruptor$clojure_handler$reify__10135.onEvent(disruptor.clj:41)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchToCursor(DisruptorQueue.java:509)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchWhenAvailable(DisruptorQueue.java:487)
	at org.apache.storm.disruptor$consume_batch_when_available.invoke(disruptor.clj:74)
	at org.apache.storm.daemon.executor$fn__10795$fn__10808$fn__10861.invoke(executor.clj:861)
	at org.apache.storm.util$async_loop$fn__553.invoke(util.clj:484)
	at clojure.lang.AFn.run(AFn.java:22)
	at java.lang.Thread.run(Thread.java:748)
15:55:27.428 [Thread-41-PrintOutBolt-executor[2 2]] ERROR c.e.d.k.s.s.PrintOutBolt - [execute,64] - Bolt的数据处理失败!数据:["Thu Feb 21 15:55:26 CST 2019"]
java.util.NoSuchElementException: null
	at java.util.ArrayList$Itr.next(ArrayList.java:854)
	at com.example.dmy.kafkastorm.storm.bolt.PrintOutBolt.execute(PrintOutBolt.java:53)
	at org.apache.storm.daemon.executor$fn__10795$tuple_action_fn__10797.invoke(executor.clj:739)
	at org.apache.storm.daemon.executor$mk_task_receiver$fn__10716.invoke(executor.clj:468)
	at org.apache.storm.disruptor$clojure_handler$reify__10135.onEvent(disruptor.clj:41)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchToCursor(DisruptorQueue.java:509)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchWhenAvailable(DisruptorQueue.java:487)
	at org.apache.storm.disruptor$consume_batch_when_available.invoke(disruptor.clj:74)
	at org.apache.storm.daemon.executor$fn__10795$fn__10808$fn__10861.invoke(executor.clj:861)
	at org.apache.storm.util$async_loop$fn__553.invoke(util.clj:484)
	at clojure.lang.AFn.run(AFn.java:22)
	at java.lang.Thread.run(Thread.java:748)
15:55:29.451 [Thread-41-PrintOutBolt-executor[2 2]] ERROR c.e.d.k.s.s.PrintOutBolt - [execute,64] - Bolt的数据处理失败!数据:["Thu Feb 21 15:55:28 CST 2019"]
java.util.NoSuchElementException: null
	at java.util.ArrayList$Itr.next(ArrayList.java:854)
	at com.example.dmy.kafkastorm.storm.bolt.PrintOutBolt.execute(PrintOutBolt.java:53)
	at org.apache.storm.daemon.executor$fn__10795$tuple_action_fn__10797.invoke(executor.clj:739)
	at org.apache.storm.daemon.executor$mk_task_receiver$fn__10716.invoke(executor.clj:468)
	at org.apache.storm.disruptor$clojure_handler$reify__10135.onEvent(disruptor.clj:41)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchToCursor(DisruptorQueue.java:509)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchWhenAvailable(DisruptorQueue.java:487)
	at org.apache.storm.disruptor$consume_batch_when_available.invoke(disruptor.clj:74)
	at org.apache.storm.daemon.executor$fn__10795$fn__10808$fn__10861.invoke(executor.clj:861)
	at org.apache.storm.util$async_loop$fn__553.invoke(util.clj:484)
	at clojure.lang.AFn.run(AFn.java:22)
	at java.lang.Thread.run(Thread.java:748)
15:55:31.470 [Thread-41-PrintOutBolt-executor[2 2]] ERROR c.e.d.k.s.s.PrintOutBolt - [execute,64] - Bolt的数据处理失败!数据:["Thu Feb 21 15:55:30 CST 2019"]
java.util.NoSuchElementException: null
	at java.util.ArrayList$Itr.next(ArrayList.java:854)
	at com.example.dmy.kafkastorm.storm.bolt.PrintOutBolt.execute(PrintOutBolt.java:53)
	at org.apache.storm.daemon.executor$fn__10795$tuple_action_fn__10797.invoke(executor.clj:739)
	at org.apache.storm.daemon.executor$mk_task_receiver$fn__10716.invoke(executor.clj:468)
	at org.apache.storm.disruptor$clojure_handler$reify__10135.onEvent(disruptor.clj:41)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchToCursor(DisruptorQueue.java:509)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchWhenAvailable(DisruptorQueue.java:487)
	at org.apache.storm.disruptor$consume_batch_when_available.invoke(disruptor.clj:74)
	at org.apache.storm.daemon.executor$fn__10795$fn__10808$fn__10861.invoke(executor.clj:861)
	at org.apache.storm.util$async_loop$fn__553.invoke(util.clj:484)
	at clojure.lang.AFn.run(AFn.java:22)
	at java.lang.Thread.run(Thread.java:748)
15:55:33.492 [Thread-41-PrintOutBolt-executor[2 2]] ERROR c.e.d.k.s.s.PrintOutBolt - [execute,64] - Bolt的数据处理失败!数据:["Thu Feb 21 15:55:32 CST 2019"]
java.util.NoSuchElementException: null
	at java.util.ArrayList$Itr.next(ArrayList.java:854)
	at com.example.dmy.kafkastorm.storm.bolt.PrintOutBolt.execute(PrintOutBolt.java:53)
	at org.apache.storm.daemon.executor$fn__10795$tuple_action_fn__10797.invoke(executor.clj:739)
	at org.apache.storm.daemon.executor$mk_task_receiver$fn__10716.invoke(executor.clj:468)
	at org.apache.storm.disruptor$clojure_handler$reify__10135.onEvent(disruptor.clj:41)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchToCursor(DisruptorQueue.java:509)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchWhenAvailable(DisruptorQueue.java:487)
	at org.apache.storm.disruptor$consume_batch_when_available.invoke(disruptor.clj:74)
	at org.apache.storm.daemon.executor$fn__10795$fn__10808$fn__10861.invoke(executor.clj:861)
	at org.apache.storm.util$async_loop$fn__553.invoke(util.clj:484)
	at clojure.lang.AFn.run(AFn.java:22)
	at java.lang.Thread.run(Thread.java:748)
15:55:35.492 [Thread-41-PrintOutBolt-executor[2 2]] ERROR c.e.d.k.s.s.PrintOutBolt - [execute,64] - Bolt的数据处理失败!数据:["Thu Feb 21 15:55:34 CST 2019"]
java.util.NoSuchElementException: null
	at java.util.ArrayList$Itr.next(ArrayList.java:854)
	at com.example.dmy.kafkastorm.storm.bolt.PrintOutBolt.execute(PrintOutBolt.java:53)
	at org.apache.storm.daemon.executor$fn__10795$tuple_action_fn__10797.invoke(executor.clj:739)
	at org.apache.storm.daemon.executor$mk_task_receiver$fn__10716.invoke(executor.clj:468)
	at org.apache.storm.disruptor$clojure_handler$reify__10135.onEvent(disruptor.clj:41)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchToCursor(DisruptorQueue.java:509)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchWhenAvailable(DisruptorQueue.java:487)
	at org.apache.storm.disruptor$consume_batch_when_available.invoke(disruptor.clj:74)
	at org.apache.storm.daemon.executor$fn__10795$fn__10808$fn__10861.invoke(executor.clj:861)
	at org.apache.storm.util$async_loop$fn__553.invoke(util.clj:484)
	at clojure.lang.AFn.run(AFn.java:22)
	at java.lang.Thread.run(Thread.java:748)
15:55:37.498 [Thread-41-PrintOutBolt-executor[2 2]] ERROR c.e.d.k.s.s.PrintOutBolt - [execute,64] - Bolt的数据处理失败!数据:["Thu Feb 21 15:55:36 CST 2019"]
java.util.NoSuchElementException: null
	at java.util.ArrayList$Itr.next(ArrayList.java:854)
	at com.example.dmy.kafkastorm.storm.bolt.PrintOutBolt.execute(PrintOutBolt.java:53)
	at org.apache.storm.daemon.executor$fn__10795$tuple_action_fn__10797.invoke(executor.clj:739)
	at org.apache.storm.daemon.executor$mk_task_receiver$fn__10716.invoke(executor.clj:468)
	at org.apache.storm.disruptor$clojure_handler$reify__10135.onEvent(disruptor.clj:41)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchToCursor(DisruptorQueue.java:509)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchWhenAvailable(DisruptorQueue.java:487)
	at org.apache.storm.disruptor$consume_batch_when_available.invoke(disruptor.clj:74)
	at org.apache.storm.daemon.executor$fn__10795$fn__10808$fn__10861.invoke(executor.clj:861)
	at org.apache.storm.util$async_loop$fn__553.invoke(util.clj:484)
	at clojure.lang.AFn.run(AFn.java:22)
	at java.lang.Thread.run(Thread.java:748)
15:55:39.497 [Thread-41-PrintOutBolt-executor[2 2]] ERROR c.e.d.k.s.s.PrintOutBolt - [execute,64] - Bolt的数据处理失败!数据:["Thu Feb 21 15:55:38 CST 2019"]
java.util.NoSuchElementException: null
	at java.util.ArrayList$Itr.next(ArrayList.java:854)
	at com.example.dmy.kafkastorm.storm.bolt.PrintOutBolt.execute(PrintOutBolt.java:53)
	at org.apache.storm.daemon.executor$fn__10795$tuple_action_fn__10797.invoke(executor.clj:739)
	at org.apache.storm.daemon.executor$mk_task_receiver$fn__10716.invoke(executor.clj:468)
	at org.apache.storm.disruptor$clojure_handler$reify__10135.onEvent(disruptor.clj:41)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchToCursor(DisruptorQueue.java:509)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchWhenAvailable(DisruptorQueue.java:487)
	at org.apache.storm.disruptor$consume_batch_when_available.invoke(disruptor.clj:74)
	at org.apache.storm.daemon.executor$fn__10795$fn__10808$fn__10861.invoke(executor.clj:861)
	at org.apache.storm.util$async_loop$fn__553.invoke(util.clj:484)
	at clojure.lang.AFn.run(AFn.java:22)
	at java.lang.Thread.run(Thread.java:748)
15:59:59.516 [Thread-41-PrintOutBolt-executor[2 2]] ERROR c.e.d.k.s.s.PrintOutBolt - [execute,64] - Bolt的数据处理失败!数据:["8"]
java.util.NoSuchElementException: null
	at java.util.ArrayList$Itr.next(ArrayList.java:854)
	at com.example.dmy.kafkastorm.storm.bolt.PrintOutBolt.execute(PrintOutBolt.java:53)
	at org.apache.storm.daemon.executor$fn__10795$tuple_action_fn__10797.invoke(executor.clj:739)
	at org.apache.storm.daemon.executor$mk_task_receiver$fn__10716.invoke(executor.clj:468)
	at org.apache.storm.disruptor$clojure_handler$reify__10135.onEvent(disruptor.clj:41)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchToCursor(DisruptorQueue.java:509)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchWhenAvailable(DisruptorQueue.java:487)
	at org.apache.storm.disruptor$consume_batch_when_available.invoke(disruptor.clj:74)
	at org.apache.storm.daemon.executor$fn__10795$fn__10808$fn__10861.invoke(executor.clj:861)
	at org.apache.storm.util$async_loop$fn__553.invoke(util.clj:484)
	at clojure.lang.AFn.run(AFn.java:22)
	at java.lang.Thread.run(Thread.java:748)
16:00:01.519 [Thread-41-PrintOutBolt-executor[2 2]] ERROR c.e.d.k.s.s.PrintOutBolt - [execute,64] - Bolt的数据处理失败!数据:["33"]
java.util.NoSuchElementException: null
	at java.util.ArrayList$Itr.next(ArrayList.java:854)
	at com.example.dmy.kafkastorm.storm.bolt.PrintOutBolt.execute(PrintOutBolt.java:53)
	at org.apache.storm.daemon.executor$fn__10795$tuple_action_fn__10797.invoke(executor.clj:739)
	at org.apache.storm.daemon.executor$mk_task_receiver$fn__10716.invoke(executor.clj:468)
	at org.apache.storm.disruptor$clojure_handler$reify__10135.onEvent(disruptor.clj:41)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchToCursor(DisruptorQueue.java:509)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchWhenAvailable(DisruptorQueue.java:487)
	at org.apache.storm.disruptor$consume_batch_when_available.invoke(disruptor.clj:74)
	at org.apache.storm.daemon.executor$fn__10795$fn__10808$fn__10861.invoke(executor.clj:861)
	at org.apache.storm.util$async_loop$fn__553.invoke(util.clj:484)
	at clojure.lang.AFn.run(AFn.java:22)
	at java.lang.Thread.run(Thread.java:748)
16:00:03.524 [Thread-41-PrintOutBolt-executor[2 2]] ERROR c.e.d.k.s.s.PrintOutBolt - [execute,64] - Bolt的数据处理失败!数据:["10"]
java.util.NoSuchElementException: null
	at java.util.ArrayList$Itr.next(ArrayList.java:854)
	at com.example.dmy.kafkastorm.storm.bolt.PrintOutBolt.execute(PrintOutBolt.java:53)
	at org.apache.storm.daemon.executor$fn__10795$tuple_action_fn__10797.invoke(executor.clj:739)
	at org.apache.storm.daemon.executor$mk_task_receiver$fn__10716.invoke(executor.clj:468)
	at org.apache.storm.disruptor$clojure_handler$reify__10135.onEvent(disruptor.clj:41)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchToCursor(DisruptorQueue.java:509)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchWhenAvailable(DisruptorQueue.java:487)
	at org.apache.storm.disruptor$consume_batch_when_available.invoke(disruptor.clj:74)
	at org.apache.storm.daemon.executor$fn__10795$fn__10808$fn__10861.invoke(executor.clj:861)
	at org.apache.storm.util$async_loop$fn__553.invoke(util.clj:484)
	at clojure.lang.AFn.run(AFn.java:22)
	at java.lang.Thread.run(Thread.java:748)
16:00:05.527 [Thread-41-PrintOutBolt-executor[2 2]] ERROR c.e.d.k.s.s.PrintOutBolt - [execute,64] - Bolt的数据处理失败!数据:["64"]
java.util.NoSuchElementException: null
	at java.util.ArrayList$Itr.next(ArrayList.java:854)
	at com.example.dmy.kafkastorm.storm.bolt.PrintOutBolt.execute(PrintOutBolt.java:53)
	at org.apache.storm.daemon.executor$fn__10795$tuple_action_fn__10797.invoke(executor.clj:739)
	at org.apache.storm.daemon.executor$mk_task_receiver$fn__10716.invoke(executor.clj:468)
	at org.apache.storm.disruptor$clojure_handler$reify__10135.onEvent(disruptor.clj:41)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchToCursor(DisruptorQueue.java:509)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchWhenAvailable(DisruptorQueue.java:487)
	at org.apache.storm.disruptor$consume_batch_when_available.invoke(disruptor.clj:74)
	at org.apache.storm.daemon.executor$fn__10795$fn__10808$fn__10861.invoke(executor.clj:861)
	at org.apache.storm.util$async_loop$fn__553.invoke(util.clj:484)
	at clojure.lang.AFn.run(AFn.java:22)
	at java.lang.Thread.run(Thread.java:748)
16:00:07.530 [Thread-41-PrintOutBolt-executor[2 2]] ERROR c.e.d.k.s.s.PrintOutBolt - [execute,64] - Bolt的数据处理失败!数据:["87"]
java.util.NoSuchElementException: null
	at java.util.ArrayList$Itr.next(ArrayList.java:854)
	at com.example.dmy.kafkastorm.storm.bolt.PrintOutBolt.execute(PrintOutBolt.java:53)
	at org.apache.storm.daemon.executor$fn__10795$tuple_action_fn__10797.invoke(executor.clj:739)
	at org.apache.storm.daemon.executor$mk_task_receiver$fn__10716.invoke(executor.clj:468)
	at org.apache.storm.disruptor$clojure_handler$reify__10135.onEvent(disruptor.clj:41)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchToCursor(DisruptorQueue.java:509)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchWhenAvailable(DisruptorQueue.java:487)
	at org.apache.storm.disruptor$consume_batch_when_available.invoke(disruptor.clj:74)
	at org.apache.storm.daemon.executor$fn__10795$fn__10808$fn__10861.invoke(executor.clj:861)
	at org.apache.storm.util$async_loop$fn__553.invoke(util.clj:484)
	at clojure.lang.AFn.run(AFn.java:22)
	at java.lang.Thread.run(Thread.java:748)
16:00:09.534 [Thread-41-PrintOutBolt-executor[2 2]] ERROR c.e.d.k.s.s.PrintOutBolt - [execute,64] - Bolt的数据处理失败!数据:["75"]
java.util.NoSuchElementException: null
	at java.util.ArrayList$Itr.next(ArrayList.java:854)
	at com.example.dmy.kafkastorm.storm.bolt.PrintOutBolt.execute(PrintOutBolt.java:53)
	at org.apache.storm.daemon.executor$fn__10795$tuple_action_fn__10797.invoke(executor.clj:739)
	at org.apache.storm.daemon.executor$mk_task_receiver$fn__10716.invoke(executor.clj:468)
	at org.apache.storm.disruptor$clojure_handler$reify__10135.onEvent(disruptor.clj:41)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchToCursor(DisruptorQueue.java:509)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchWhenAvailable(DisruptorQueue.java:487)
	at org.apache.storm.disruptor$consume_batch_when_available.invoke(disruptor.clj:74)
	at org.apache.storm.daemon.executor$fn__10795$fn__10808$fn__10861.invoke(executor.clj:861)
	at org.apache.storm.util$async_loop$fn__553.invoke(util.clj:484)
	at clojure.lang.AFn.run(AFn.java:22)
	at java.lang.Thread.run(Thread.java:748)
16:00:11.538 [Thread-41-PrintOutBolt-executor[2 2]] ERROR c.e.d.k.s.s.PrintOutBolt - [execute,64] - Bolt的数据处理失败!数据:["68"]
java.util.NoSuchElementException: null
	at java.util.ArrayList$Itr.next(ArrayList.java:854)
	at com.example.dmy.kafkastorm.storm.bolt.PrintOutBolt.execute(PrintOutBolt.java:53)
	at org.apache.storm.daemon.executor$fn__10795$tuple_action_fn__10797.invoke(executor.clj:739)
	at org.apache.storm.daemon.executor$mk_task_receiver$fn__10716.invoke(executor.clj:468)
	at org.apache.storm.disruptor$clojure_handler$reify__10135.onEvent(disruptor.clj:41)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchToCursor(DisruptorQueue.java:509)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchWhenAvailable(DisruptorQueue.java:487)
	at org.apache.storm.disruptor$consume_batch_when_available.invoke(disruptor.clj:74)
	at org.apache.storm.daemon.executor$fn__10795$fn__10808$fn__10861.invoke(executor.clj:861)
	at org.apache.storm.util$async_loop$fn__553.invoke(util.clj:484)
	at clojure.lang.AFn.run(AFn.java:22)
	at java.lang.Thread.run(Thread.java:748)
16:00:13.541 [Thread-41-PrintOutBolt-executor[2 2]] ERROR c.e.d.k.s.s.PrintOutBolt - [execute,64] - Bolt的数据处理失败!数据:["22"]
java.util.NoSuchElementException: null
	at java.util.ArrayList$Itr.next(ArrayList.java:854)
	at com.example.dmy.kafkastorm.storm.bolt.PrintOutBolt.execute(PrintOutBolt.java:53)
	at org.apache.storm.daemon.executor$fn__10795$tuple_action_fn__10797.invoke(executor.clj:739)
	at org.apache.storm.daemon.executor$mk_task_receiver$fn__10716.invoke(executor.clj:468)
	at org.apache.storm.disruptor$clojure_handler$reify__10135.onEvent(disruptor.clj:41)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchToCursor(DisruptorQueue.java:509)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchWhenAvailable(DisruptorQueue.java:487)
	at org.apache.storm.disruptor$consume_batch_when_available.invoke(disruptor.clj:74)
	at org.apache.storm.daemon.executor$fn__10795$fn__10808$fn__10861.invoke(executor.clj:861)
	at org.apache.storm.util$async_loop$fn__553.invoke(util.clj:484)
	at clojure.lang.AFn.run(AFn.java:22)
	at java.lang.Thread.run(Thread.java:748)
16:00:15.542 [Thread-41-PrintOutBolt-executor[2 2]] ERROR c.e.d.k.s.s.PrintOutBolt - [execute,64] - Bolt的数据处理失败!数据:["8"]
java.util.NoSuchElementException: null
	at java.util.ArrayList$Itr.next(ArrayList.java:854)
	at com.example.dmy.kafkastorm.storm.bolt.PrintOutBolt.execute(PrintOutBolt.java:53)
	at org.apache.storm.daemon.executor$fn__10795$tuple_action_fn__10797.invoke(executor.clj:739)
	at org.apache.storm.daemon.executor$mk_task_receiver$fn__10716.invoke(executor.clj:468)
	at org.apache.storm.disruptor$clojure_handler$reify__10135.onEvent(disruptor.clj:41)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchToCursor(DisruptorQueue.java:509)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchWhenAvailable(DisruptorQueue.java:487)
	at org.apache.storm.disruptor$consume_batch_when_available.invoke(disruptor.clj:74)
	at org.apache.storm.daemon.executor$fn__10795$fn__10808$fn__10861.invoke(executor.clj:861)
	at org.apache.storm.util$async_loop$fn__553.invoke(util.clj:484)
	at clojure.lang.AFn.run(AFn.java:22)
	at java.lang.Thread.run(Thread.java:748)
16:00:17.547 [Thread-41-PrintOutBolt-executor[2 2]] ERROR c.e.d.k.s.s.PrintOutBolt - [execute,64] - Bolt的数据处理失败!数据:["80"]
java.util.NoSuchElementException: null
	at java.util.ArrayList$Itr.next(ArrayList.java:854)
	at com.example.dmy.kafkastorm.storm.bolt.PrintOutBolt.execute(PrintOutBolt.java:53)
	at org.apache.storm.daemon.executor$fn__10795$tuple_action_fn__10797.invoke(executor.clj:739)
	at org.apache.storm.daemon.executor$mk_task_receiver$fn__10716.invoke(executor.clj:468)
	at org.apache.storm.disruptor$clojure_handler$reify__10135.onEvent(disruptor.clj:41)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchToCursor(DisruptorQueue.java:509)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchWhenAvailable(DisruptorQueue.java:487)
	at org.apache.storm.disruptor$consume_batch_when_available.invoke(disruptor.clj:74)
	at org.apache.storm.daemon.executor$fn__10795$fn__10808$fn__10861.invoke(executor.clj:861)
	at org.apache.storm.util$async_loop$fn__553.invoke(util.clj:484)
	at clojure.lang.AFn.run(AFn.java:22)
	at java.lang.Thread.run(Thread.java:748)
16:00:19.550 [Thread-41-PrintOutBolt-executor[2 2]] ERROR c.e.d.k.s.s.PrintOutBolt - [execute,64] - Bolt的数据处理失败!数据:["96"]
java.util.NoSuchElementException: null
	at java.util.ArrayList$Itr.next(ArrayList.java:854)
	at com.example.dmy.kafkastorm.storm.bolt.PrintOutBolt.execute(PrintOutBolt.java:53)
	at org.apache.storm.daemon.executor$fn__10795$tuple_action_fn__10797.invoke(executor.clj:739)
	at org.apache.storm.daemon.executor$mk_task_receiver$fn__10716.invoke(executor.clj:468)
	at org.apache.storm.disruptor$clojure_handler$reify__10135.onEvent(disruptor.clj:41)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchToCursor(DisruptorQueue.java:509)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchWhenAvailable(DisruptorQueue.java:487)
	at org.apache.storm.disruptor$consume_batch_when_available.invoke(disruptor.clj:74)
	at org.apache.storm.daemon.executor$fn__10795$fn__10808$fn__10861.invoke(executor.clj:861)
	at org.apache.storm.util$async_loop$fn__553.invoke(util.clj:484)
	at clojure.lang.AFn.run(AFn.java:22)
	at java.lang.Thread.run(Thread.java:748)
16:00:22.700 [Thread-41-PrintOutBolt-executor[2 2]] ERROR c.e.d.k.s.s.PrintOutBolt - [execute,64] - Bolt的数据处理失败!数据:["93"]
java.util.NoSuchElementException: null
	at java.util.ArrayList$Itr.next(ArrayList.java:854)
	at com.example.dmy.kafkastorm.storm.bolt.PrintOutBolt.execute(PrintOutBolt.java:53)
	at org.apache.storm.daemon.executor$fn__10795$tuple_action_fn__10797.invoke(executor.clj:739)
	at org.apache.storm.daemon.executor$mk_task_receiver$fn__10716.invoke(executor.clj:468)
	at org.apache.storm.disruptor$clojure_handler$reify__10135.onEvent(disruptor.clj:41)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchToCursor(DisruptorQueue.java:509)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchWhenAvailable(DisruptorQueue.java:487)
	at org.apache.storm.disruptor$consume_batch_when_available.invoke(disruptor.clj:74)
	at org.apache.storm.daemon.executor$fn__10795$fn__10808$fn__10861.invoke(executor.clj:861)
	at org.apache.storm.util$async_loop$fn__553.invoke(util.clj:484)
	at clojure.lang.AFn.run(AFn.java:22)
	at java.lang.Thread.run(Thread.java:748)
16:00:24.672 [Thread-41-PrintOutBolt-executor[2 2]] ERROR c.e.d.k.s.s.PrintOutBolt - [execute,64] - Bolt的数据处理失败!数据:["11"]
java.util.NoSuchElementException: null
	at java.util.ArrayList$Itr.next(ArrayList.java:854)
	at com.example.dmy.kafkastorm.storm.bolt.PrintOutBolt.execute(PrintOutBolt.java:53)
	at org.apache.storm.daemon.executor$fn__10795$tuple_action_fn__10797.invoke(executor.clj:739)
	at org.apache.storm.daemon.executor$mk_task_receiver$fn__10716.invoke(executor.clj:468)
	at org.apache.storm.disruptor$clojure_handler$reify__10135.onEvent(disruptor.clj:41)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchToCursor(DisruptorQueue.java:509)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchWhenAvailable(DisruptorQueue.java:487)
	at org.apache.storm.disruptor$consume_batch_when_available.invoke(disruptor.clj:74)
	at org.apache.storm.daemon.executor$fn__10795$fn__10808$fn__10861.invoke(executor.clj:861)
	at org.apache.storm.util$async_loop$fn__553.invoke(util.clj:484)
	at clojure.lang.AFn.run(AFn.java:22)
	at java.lang.Thread.run(Thread.java:748)
16:00:26.675 [Thread-41-PrintOutBolt-executor[2 2]] ERROR c.e.d.k.s.s.PrintOutBolt - [execute,64] - Bolt的数据处理失败!数据:["83"]
java.util.NoSuchElementException: null
	at java.util.ArrayList$Itr.next(ArrayList.java:854)
	at com.example.dmy.kafkastorm.storm.bolt.PrintOutBolt.execute(PrintOutBolt.java:53)
	at org.apache.storm.daemon.executor$fn__10795$tuple_action_fn__10797.invoke(executor.clj:739)
	at org.apache.storm.daemon.executor$mk_task_receiver$fn__10716.invoke(executor.clj:468)
	at org.apache.storm.disruptor$clojure_handler$reify__10135.onEvent(disruptor.clj:41)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchToCursor(DisruptorQueue.java:509)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchWhenAvailable(DisruptorQueue.java:487)
	at org.apache.storm.disruptor$consume_batch_when_available.invoke(disruptor.clj:74)
	at org.apache.storm.daemon.executor$fn__10795$fn__10808$fn__10861.invoke(executor.clj:861)
	at org.apache.storm.util$async_loop$fn__553.invoke(util.clj:484)
	at clojure.lang.AFn.run(AFn.java:22)
	at java.lang.Thread.run(Thread.java:748)
16:00:28.677 [Thread-41-PrintOutBolt-executor[2 2]] ERROR c.e.d.k.s.s.PrintOutBolt - [execute,64] - Bolt的数据处理失败!数据:["37"]
java.util.NoSuchElementException: null
	at java.util.ArrayList$Itr.next(ArrayList.java:854)
	at com.example.dmy.kafkastorm.storm.bolt.PrintOutBolt.execute(PrintOutBolt.java:53)
	at org.apache.storm.daemon.executor$fn__10795$tuple_action_fn__10797.invoke(executor.clj:739)
	at org.apache.storm.daemon.executor$mk_task_receiver$fn__10716.invoke(executor.clj:468)
	at org.apache.storm.disruptor$clojure_handler$reify__10135.onEvent(disruptor.clj:41)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchToCursor(DisruptorQueue.java:509)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchWhenAvailable(DisruptorQueue.java:487)
	at org.apache.storm.disruptor$consume_batch_when_available.invoke(disruptor.clj:74)
	at org.apache.storm.daemon.executor$fn__10795$fn__10808$fn__10861.invoke(executor.clj:861)
	at org.apache.storm.util$async_loop$fn__553.invoke(util.clj:484)
	at clojure.lang.AFn.run(AFn.java:22)
	at java.lang.Thread.run(Thread.java:748)
16:00:30.680 [Thread-41-PrintOutBolt-executor[2 2]] ERROR c.e.d.k.s.s.PrintOutBolt - [execute,64] - Bolt的数据处理失败!数据:["27"]
java.util.NoSuchElementException: null
	at java.util.ArrayList$Itr.next(ArrayList.java:854)
	at com.example.dmy.kafkastorm.storm.bolt.PrintOutBolt.execute(PrintOutBolt.java:53)
	at org.apache.storm.daemon.executor$fn__10795$tuple_action_fn__10797.invoke(executor.clj:739)
	at org.apache.storm.daemon.executor$mk_task_receiver$fn__10716.invoke(executor.clj:468)
	at org.apache.storm.disruptor$clojure_handler$reify__10135.onEvent(disruptor.clj:41)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchToCursor(DisruptorQueue.java:509)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchWhenAvailable(DisruptorQueue.java:487)
	at org.apache.storm.disruptor$consume_batch_when_available.invoke(disruptor.clj:74)
	at org.apache.storm.daemon.executor$fn__10795$fn__10808$fn__10861.invoke(executor.clj:861)
	at org.apache.storm.util$async_loop$fn__553.invoke(util.clj:484)
	at clojure.lang.AFn.run(AFn.java:22)
	at java.lang.Thread.run(Thread.java:748)
16:00:32.683 [Thread-41-PrintOutBolt-executor[2 2]] ERROR c.e.d.k.s.s.PrintOutBolt - [execute,64] - Bolt的数据处理失败!数据:["99"]
java.util.NoSuchElementException: null
	at java.util.ArrayList$Itr.next(ArrayList.java:854)
	at com.example.dmy.kafkastorm.storm.bolt.PrintOutBolt.execute(PrintOutBolt.java:53)
	at org.apache.storm.daemon.executor$fn__10795$tuple_action_fn__10797.invoke(executor.clj:739)
	at org.apache.storm.daemon.executor$mk_task_receiver$fn__10716.invoke(executor.clj:468)
	at org.apache.storm.disruptor$clojure_handler$reify__10135.onEvent(disruptor.clj:41)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchToCursor(DisruptorQueue.java:509)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchWhenAvailable(DisruptorQueue.java:487)
	at org.apache.storm.disruptor$consume_batch_when_available.invoke(disruptor.clj:74)
	at org.apache.storm.daemon.executor$fn__10795$fn__10808$fn__10861.invoke(executor.clj:861)
	at org.apache.storm.util$async_loop$fn__553.invoke(util.clj:484)
	at clojure.lang.AFn.run(AFn.java:22)
	at java.lang.Thread.run(Thread.java:748)
16:00:34.687 [Thread-41-PrintOutBolt-executor[2 2]] ERROR c.e.d.k.s.s.PrintOutBolt - [execute,64] - Bolt的数据处理失败!数据:["80"]
java.util.NoSuchElementException: null
	at java.util.ArrayList$Itr.next(ArrayList.java:854)
	at com.example.dmy.kafkastorm.storm.bolt.PrintOutBolt.execute(PrintOutBolt.java:53)
	at org.apache.storm.daemon.executor$fn__10795$tuple_action_fn__10797.invoke(executor.clj:739)
	at org.apache.storm.daemon.executor$mk_task_receiver$fn__10716.invoke(executor.clj:468)
	at org.apache.storm.disruptor$clojure_handler$reify__10135.onEvent(disruptor.clj:41)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchToCursor(DisruptorQueue.java:509)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchWhenAvailable(DisruptorQueue.java:487)
	at org.apache.storm.disruptor$consume_batch_when_available.invoke(disruptor.clj:74)
	at org.apache.storm.daemon.executor$fn__10795$fn__10808$fn__10861.invoke(executor.clj:861)
	at org.apache.storm.util$async_loop$fn__553.invoke(util.clj:484)
	at clojure.lang.AFn.run(AFn.java:22)
	at java.lang.Thread.run(Thread.java:748)
16:00:36.689 [Thread-41-PrintOutBolt-executor[2 2]] ERROR c.e.d.k.s.s.PrintOutBolt - [execute,64] - Bolt的数据处理失败!数据:["33"]
java.util.NoSuchElementException: null
	at java.util.ArrayList$Itr.next(ArrayList.java:854)
	at com.example.dmy.kafkastorm.storm.bolt.PrintOutBolt.execute(PrintOutBolt.java:53)
	at org.apache.storm.daemon.executor$fn__10795$tuple_action_fn__10797.invoke(executor.clj:739)
	at org.apache.storm.daemon.executor$mk_task_receiver$fn__10716.invoke(executor.clj:468)
	at org.apache.storm.disruptor$clojure_handler$reify__10135.onEvent(disruptor.clj:41)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchToCursor(DisruptorQueue.java:509)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchWhenAvailable(DisruptorQueue.java:487)
	at org.apache.storm.disruptor$consume_batch_when_available.invoke(disruptor.clj:74)
	at org.apache.storm.daemon.executor$fn__10795$fn__10808$fn__10861.invoke(executor.clj:861)
	at org.apache.storm.util$async_loop$fn__553.invoke(util.clj:484)
	at clojure.lang.AFn.run(AFn.java:22)
	at java.lang.Thread.run(Thread.java:748)
16:00:38.693 [Thread-41-PrintOutBolt-executor[2 2]] ERROR c.e.d.k.s.s.PrintOutBolt - [execute,64] - Bolt的数据处理失败!数据:["49"]
java.util.NoSuchElementException: null
	at java.util.ArrayList$Itr.next(ArrayList.java:854)
	at com.example.dmy.kafkastorm.storm.bolt.PrintOutBolt.execute(PrintOutBolt.java:53)
	at org.apache.storm.daemon.executor$fn__10795$tuple_action_fn__10797.invoke(executor.clj:739)
	at org.apache.storm.daemon.executor$mk_task_receiver$fn__10716.invoke(executor.clj:468)
	at org.apache.storm.disruptor$clojure_handler$reify__10135.onEvent(disruptor.clj:41)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchToCursor(DisruptorQueue.java:509)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchWhenAvailable(DisruptorQueue.java:487)
	at org.apache.storm.disruptor$consume_batch_when_available.invoke(disruptor.clj:74)
	at org.apache.storm.daemon.executor$fn__10795$fn__10808$fn__10861.invoke(executor.clj:861)
	at org.apache.storm.util$async_loop$fn__553.invoke(util.clj:484)
	at clojure.lang.AFn.run(AFn.java:22)
	at java.lang.Thread.run(Thread.java:748)
16:02:17.408 [Thread-40-PrintOutBolt-executor[2 2]] ERROR c.e.d.k.s.s.PrintOutBolt - [execute,64] - Bolt的数据处理失败!数据:["45","1","6","7","7","8","8","5","4","2","6","4","9"]
java.util.NoSuchElementException: null
	at java.util.ArrayList$Itr.next(ArrayList.java:854)
	at com.example.dmy.kafkastorm.storm.bolt.PrintOutBolt.execute(PrintOutBolt.java:53)
	at org.apache.storm.daemon.executor$fn__10795$tuple_action_fn__10797.invoke(executor.clj:739)
	at org.apache.storm.daemon.executor$mk_task_receiver$fn__10716.invoke(executor.clj:468)
	at org.apache.storm.disruptor$clojure_handler$reify__10135.onEvent(disruptor.clj:41)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchToCursor(DisruptorQueue.java:509)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchWhenAvailable(DisruptorQueue.java:487)
	at org.apache.storm.disruptor$consume_batch_when_available.invoke(disruptor.clj:74)
	at org.apache.storm.daemon.executor$fn__10795$fn__10808$fn__10861.invoke(executor.clj:861)
	at org.apache.storm.util$async_loop$fn__553.invoke(util.clj:484)
	at clojure.lang.AFn.run(AFn.java:22)
	at java.lang.Thread.run(Thread.java:748)
16:02:19.402 [Thread-40-PrintOutBolt-executor[2 2]] ERROR c.e.d.k.s.s.PrintOutBolt - [execute,64] - Bolt的数据处理失败!数据:["1"]
java.util.NoSuchElementException: null
	at java.util.ArrayList$Itr.next(ArrayList.java:854)
	at com.example.dmy.kafkastorm.storm.bolt.PrintOutBolt.execute(PrintOutBolt.java:53)
	at org.apache.storm.daemon.executor$fn__10795$tuple_action_fn__10797.invoke(executor.clj:739)
	at org.apache.storm.daemon.executor$mk_task_receiver$fn__10716.invoke(executor.clj:468)
	at org.apache.storm.disruptor$clojure_handler$reify__10135.onEvent(disruptor.clj:41)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchToCursor(DisruptorQueue.java:509)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchWhenAvailable(DisruptorQueue.java:487)
	at org.apache.storm.disruptor$consume_batch_when_available.invoke(disruptor.clj:74)
	at org.apache.storm.daemon.executor$fn__10795$fn__10808$fn__10861.invoke(executor.clj:861)
	at org.apache.storm.util$async_loop$fn__553.invoke(util.clj:484)
	at clojure.lang.AFn.run(AFn.java:22)
	at java.lang.Thread.run(Thread.java:748)
16:02:21.426 [Thread-40-PrintOutBolt-executor[2 2]] ERROR c.e.d.k.s.s.PrintOutBolt - [execute,64] - Bolt的数据处理失败!数据:["8"]
java.util.NoSuchElementException: null
	at java.util.ArrayList$Itr.next(ArrayList.java:854)
	at com.example.dmy.kafkastorm.storm.bolt.PrintOutBolt.execute(PrintOutBolt.java:53)
	at org.apache.storm.daemon.executor$fn__10795$tuple_action_fn__10797.invoke(executor.clj:739)
	at org.apache.storm.daemon.executor$mk_task_receiver$fn__10716.invoke(executor.clj:468)
	at org.apache.storm.disruptor$clojure_handler$reify__10135.onEvent(disruptor.clj:41)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchToCursor(DisruptorQueue.java:509)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchWhenAvailable(DisruptorQueue.java:487)
	at org.apache.storm.disruptor$consume_batch_when_available.invoke(disruptor.clj:74)
	at org.apache.storm.daemon.executor$fn__10795$fn__10808$fn__10861.invoke(executor.clj:861)
	at org.apache.storm.util$async_loop$fn__553.invoke(util.clj:484)
	at clojure.lang.AFn.run(AFn.java:22)
	at java.lang.Thread.run(Thread.java:748)
16:02:23.427 [Thread-40-PrintOutBolt-executor[2 2]] ERROR c.e.d.k.s.s.PrintOutBolt - [execute,64] - Bolt的数据处理失败!数据:["9"]
java.util.NoSuchElementException: null
	at java.util.ArrayList$Itr.next(ArrayList.java:854)
	at com.example.dmy.kafkastorm.storm.bolt.PrintOutBolt.execute(PrintOutBolt.java:53)
	at org.apache.storm.daemon.executor$fn__10795$tuple_action_fn__10797.invoke(executor.clj:739)
	at org.apache.storm.daemon.executor$mk_task_receiver$fn__10716.invoke(executor.clj:468)
	at org.apache.storm.disruptor$clojure_handler$reify__10135.onEvent(disruptor.clj:41)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchToCursor(DisruptorQueue.java:509)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchWhenAvailable(DisruptorQueue.java:487)
	at org.apache.storm.disruptor$consume_batch_when_available.invoke(disruptor.clj:74)
	at org.apache.storm.daemon.executor$fn__10795$fn__10808$fn__10861.invoke(executor.clj:861)
	at org.apache.storm.util$async_loop$fn__553.invoke(util.clj:484)
	at clojure.lang.AFn.run(AFn.java:22)
	at java.lang.Thread.run(Thread.java:748)
16:02:25.451 [Thread-40-PrintOutBolt-executor[2 2]] ERROR c.e.d.k.s.s.PrintOutBolt - [execute,64] - Bolt的数据处理失败!数据:["5"]
java.util.NoSuchElementException: null
	at java.util.ArrayList$Itr.next(ArrayList.java:854)
	at com.example.dmy.kafkastorm.storm.bolt.PrintOutBolt.execute(PrintOutBolt.java:53)
	at org.apache.storm.daemon.executor$fn__10795$tuple_action_fn__10797.invoke(executor.clj:739)
	at org.apache.storm.daemon.executor$mk_task_receiver$fn__10716.invoke(executor.clj:468)
	at org.apache.storm.disruptor$clojure_handler$reify__10135.onEvent(disruptor.clj:41)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchToCursor(DisruptorQueue.java:509)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchWhenAvailable(DisruptorQueue.java:487)
	at org.apache.storm.disruptor$consume_batch_when_available.invoke(disruptor.clj:74)
	at org.apache.storm.daemon.executor$fn__10795$fn__10808$fn__10861.invoke(executor.clj:861)
	at org.apache.storm.util$async_loop$fn__553.invoke(util.clj:484)
	at clojure.lang.AFn.run(AFn.java:22)
	at java.lang.Thread.run(Thread.java:748)
16:02:27.452 [Thread-40-PrintOutBolt-executor[2 2]] ERROR c.e.d.k.s.s.PrintOutBolt - [execute,64] - Bolt的数据处理失败!数据:["1"]
java.util.NoSuchElementException: null
	at java.util.ArrayList$Itr.next(ArrayList.java:854)
	at com.example.dmy.kafkastorm.storm.bolt.PrintOutBolt.execute(PrintOutBolt.java:53)
	at org.apache.storm.daemon.executor$fn__10795$tuple_action_fn__10797.invoke(executor.clj:739)
	at org.apache.storm.daemon.executor$mk_task_receiver$fn__10716.invoke(executor.clj:468)
	at org.apache.storm.disruptor$clojure_handler$reify__10135.onEvent(disruptor.clj:41)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchToCursor(DisruptorQueue.java:509)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchWhenAvailable(DisruptorQueue.java:487)
	at org.apache.storm.disruptor$consume_batch_when_available.invoke(disruptor.clj:74)
	at org.apache.storm.daemon.executor$fn__10795$fn__10808$fn__10861.invoke(executor.clj:861)
	at org.apache.storm.util$async_loop$fn__553.invoke(util.clj:484)
	at clojure.lang.AFn.run(AFn.java:22)
	at java.lang.Thread.run(Thread.java:748)
16:02:29.481 [Thread-40-PrintOutBolt-executor[2 2]] ERROR c.e.d.k.s.s.PrintOutBolt - [execute,64] - Bolt的数据处理失败!数据:["9"]
java.util.NoSuchElementException: null
	at java.util.ArrayList$Itr.next(ArrayList.java:854)
	at com.example.dmy.kafkastorm.storm.bolt.PrintOutBolt.execute(PrintOutBolt.java:53)
	at org.apache.storm.daemon.executor$fn__10795$tuple_action_fn__10797.invoke(executor.clj:739)
	at org.apache.storm.daemon.executor$mk_task_receiver$fn__10716.invoke(executor.clj:468)
	at org.apache.storm.disruptor$clojure_handler$reify__10135.onEvent(disruptor.clj:41)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchToCursor(DisruptorQueue.java:509)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchWhenAvailable(DisruptorQueue.java:487)
	at org.apache.storm.disruptor$consume_batch_when_available.invoke(disruptor.clj:74)
	at org.apache.storm.daemon.executor$fn__10795$fn__10808$fn__10861.invoke(executor.clj:861)
	at org.apache.storm.util$async_loop$fn__553.invoke(util.clj:484)
	at clojure.lang.AFn.run(AFn.java:22)
	at java.lang.Thread.run(Thread.java:748)
16:02:31.483 [Thread-40-PrintOutBolt-executor[2 2]] ERROR c.e.d.k.s.s.PrintOutBolt - [execute,64] - Bolt的数据处理失败!数据:["2"]
java.util.NoSuchElementException: null
	at java.util.ArrayList$Itr.next(ArrayList.java:854)
	at com.example.dmy.kafkastorm.storm.bolt.PrintOutBolt.execute(PrintOutBolt.java:53)
	at org.apache.storm.daemon.executor$fn__10795$tuple_action_fn__10797.invoke(executor.clj:739)
	at org.apache.storm.daemon.executor$mk_task_receiver$fn__10716.invoke(executor.clj:468)
	at org.apache.storm.disruptor$clojure_handler$reify__10135.onEvent(disruptor.clj:41)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchToCursor(DisruptorQueue.java:509)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchWhenAvailable(DisruptorQueue.java:487)
	at org.apache.storm.disruptor$consume_batch_when_available.invoke(disruptor.clj:74)
	at org.apache.storm.daemon.executor$fn__10795$fn__10808$fn__10861.invoke(executor.clj:861)
	at org.apache.storm.util$async_loop$fn__553.invoke(util.clj:484)
	at clojure.lang.AFn.run(AFn.java:22)
	at java.lang.Thread.run(Thread.java:748)
16:02:33.506 [Thread-40-PrintOutBolt-executor[2 2]] ERROR c.e.d.k.s.s.PrintOutBolt - [execute,64] - Bolt的数据处理失败!数据:["5"]
java.util.NoSuchElementException: null
	at java.util.ArrayList$Itr.next(ArrayList.java:854)
	at com.example.dmy.kafkastorm.storm.bolt.PrintOutBolt.execute(PrintOutBolt.java:53)
	at org.apache.storm.daemon.executor$fn__10795$tuple_action_fn__10797.invoke(executor.clj:739)
	at org.apache.storm.daemon.executor$mk_task_receiver$fn__10716.invoke(executor.clj:468)
	at org.apache.storm.disruptor$clojure_handler$reify__10135.onEvent(disruptor.clj:41)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchToCursor(DisruptorQueue.java:509)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchWhenAvailable(DisruptorQueue.java:487)
	at org.apache.storm.disruptor$consume_batch_when_available.invoke(disruptor.clj:74)
	at org.apache.storm.daemon.executor$fn__10795$fn__10808$fn__10861.invoke(executor.clj:861)
	at org.apache.storm.util$async_loop$fn__553.invoke(util.clj:484)
	at clojure.lang.AFn.run(AFn.java:22)
	at java.lang.Thread.run(Thread.java:748)
16:02:35.509 [Thread-40-PrintOutBolt-executor[2 2]] ERROR c.e.d.k.s.s.PrintOutBolt - [execute,64] - Bolt的数据处理失败!数据:["4"]
java.util.NoSuchElementException: null
	at java.util.ArrayList$Itr.next(ArrayList.java:854)
	at com.example.dmy.kafkastorm.storm.bolt.PrintOutBolt.execute(PrintOutBolt.java:53)
	at org.apache.storm.daemon.executor$fn__10795$tuple_action_fn__10797.invoke(executor.clj:739)
	at org.apache.storm.daemon.executor$mk_task_receiver$fn__10716.invoke(executor.clj:468)
	at org.apache.storm.disruptor$clojure_handler$reify__10135.onEvent(disruptor.clj:41)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchToCursor(DisruptorQueue.java:509)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchWhenAvailable(DisruptorQueue.java:487)
	at org.apache.storm.disruptor$consume_batch_when_available.invoke(disruptor.clj:74)
	at org.apache.storm.daemon.executor$fn__10795$fn__10808$fn__10861.invoke(executor.clj:861)
	at org.apache.storm.util$async_loop$fn__553.invoke(util.clj:484)
	at clojure.lang.AFn.run(AFn.java:22)
	at java.lang.Thread.run(Thread.java:748)
16:02:37.514 [Thread-40-PrintOutBolt-executor[2 2]] ERROR c.e.d.k.s.s.PrintOutBolt - [execute,64] - Bolt的数据处理失败!数据:["7"]
java.util.NoSuchElementException: null
	at java.util.ArrayList$Itr.next(ArrayList.java:854)
	at com.example.dmy.kafkastorm.storm.bolt.PrintOutBolt.execute(PrintOutBolt.java:53)
	at org.apache.storm.daemon.executor$fn__10795$tuple_action_fn__10797.invoke(executor.clj:739)
	at org.apache.storm.daemon.executor$mk_task_receiver$fn__10716.invoke(executor.clj:468)
	at org.apache.storm.disruptor$clojure_handler$reify__10135.onEvent(disruptor.clj:41)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchToCursor(DisruptorQueue.java:509)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchWhenAvailable(DisruptorQueue.java:487)
	at org.apache.storm.disruptor$consume_batch_when_available.invoke(disruptor.clj:74)
	at org.apache.storm.daemon.executor$fn__10795$fn__10808$fn__10861.invoke(executor.clj:861)
	at org.apache.storm.util$async_loop$fn__553.invoke(util.clj:484)
	at clojure.lang.AFn.run(AFn.java:22)
	at java.lang.Thread.run(Thread.java:748)
16:02:39.515 [Thread-40-PrintOutBolt-executor[2 2]] ERROR c.e.d.k.s.s.PrintOutBolt - [execute,64] - Bolt的数据处理失败!数据:["4"]
java.util.NoSuchElementException: null
	at java.util.ArrayList$Itr.next(ArrayList.java:854)
	at com.example.dmy.kafkastorm.storm.bolt.PrintOutBolt.execute(PrintOutBolt.java:53)
	at org.apache.storm.daemon.executor$fn__10795$tuple_action_fn__10797.invoke(executor.clj:739)
	at org.apache.storm.daemon.executor$mk_task_receiver$fn__10716.invoke(executor.clj:468)
	at org.apache.storm.disruptor$clojure_handler$reify__10135.onEvent(disruptor.clj:41)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchToCursor(DisruptorQueue.java:509)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchWhenAvailable(DisruptorQueue.java:487)
	at org.apache.storm.disruptor$consume_batch_when_available.invoke(disruptor.clj:74)
	at org.apache.storm.daemon.executor$fn__10795$fn__10808$fn__10861.invoke(executor.clj:861)
	at org.apache.storm.util$async_loop$fn__553.invoke(util.clj:484)
	at clojure.lang.AFn.run(AFn.java:22)
	at java.lang.Thread.run(Thread.java:748)
16:02:42.926 [Thread-40-PrintOutBolt-executor[2 2]] ERROR c.e.d.k.s.s.PrintOutBolt - [execute,64] - Bolt的数据处理失败!数据:["3"]
java.util.NoSuchElementException: null
	at java.util.ArrayList$Itr.next(ArrayList.java:854)
	at com.example.dmy.kafkastorm.storm.bolt.PrintOutBolt.execute(PrintOutBolt.java:53)
	at org.apache.storm.daemon.executor$fn__10795$tuple_action_fn__10797.invoke(executor.clj:739)
	at org.apache.storm.daemon.executor$mk_task_receiver$fn__10716.invoke(executor.clj:468)
	at org.apache.storm.disruptor$clojure_handler$reify__10135.onEvent(disruptor.clj:41)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchToCursor(DisruptorQueue.java:509)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchWhenAvailable(DisruptorQueue.java:487)
	at org.apache.storm.disruptor$consume_batch_when_available.invoke(disruptor.clj:74)
	at org.apache.storm.daemon.executor$fn__10795$fn__10808$fn__10861.invoke(executor.clj:861)
	at org.apache.storm.util$async_loop$fn__553.invoke(util.clj:484)
	at clojure.lang.AFn.run(AFn.java:22)
	at java.lang.Thread.run(Thread.java:748)
16:02:44.793 [Thread-40-PrintOutBolt-executor[2 2]] ERROR c.e.d.k.s.s.PrintOutBolt - [execute,64] - Bolt的数据处理失败!数据:["6"]
java.util.NoSuchElementException: null
	at java.util.ArrayList$Itr.next(ArrayList.java:854)
	at com.example.dmy.kafkastorm.storm.bolt.PrintOutBolt.execute(PrintOutBolt.java:53)
	at org.apache.storm.daemon.executor$fn__10795$tuple_action_fn__10797.invoke(executor.clj:739)
	at org.apache.storm.daemon.executor$mk_task_receiver$fn__10716.invoke(executor.clj:468)
	at org.apache.storm.disruptor$clojure_handler$reify__10135.onEvent(disruptor.clj:41)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchToCursor(DisruptorQueue.java:509)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchWhenAvailable(DisruptorQueue.java:487)
	at org.apache.storm.disruptor$consume_batch_when_available.invoke(disruptor.clj:74)
	at org.apache.storm.daemon.executor$fn__10795$fn__10808$fn__10861.invoke(executor.clj:861)
	at org.apache.storm.util$async_loop$fn__553.invoke(util.clj:484)
	at clojure.lang.AFn.run(AFn.java:22)
	at java.lang.Thread.run(Thread.java:748)
16:02:46.796 [Thread-40-PrintOutBolt-executor[2 2]] ERROR c.e.d.k.s.s.PrintOutBolt - [execute,64] - Bolt的数据处理失败!数据:["1"]
java.util.NoSuchElementException: null
	at java.util.ArrayList$Itr.next(ArrayList.java:854)
	at com.example.dmy.kafkastorm.storm.bolt.PrintOutBolt.execute(PrintOutBolt.java:53)
	at org.apache.storm.daemon.executor$fn__10795$tuple_action_fn__10797.invoke(executor.clj:739)
	at org.apache.storm.daemon.executor$mk_task_receiver$fn__10716.invoke(executor.clj:468)
	at org.apache.storm.disruptor$clojure_handler$reify__10135.onEvent(disruptor.clj:41)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchToCursor(DisruptorQueue.java:509)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchWhenAvailable(DisruptorQueue.java:487)
	at org.apache.storm.disruptor$consume_batch_when_available.invoke(disruptor.clj:74)
	at org.apache.storm.daemon.executor$fn__10795$fn__10808$fn__10861.invoke(executor.clj:861)
	at org.apache.storm.util$async_loop$fn__553.invoke(util.clj:484)
	at clojure.lang.AFn.run(AFn.java:22)
	at java.lang.Thread.run(Thread.java:748)
16:02:48.805 [Thread-40-PrintOutBolt-executor[2 2]] ERROR c.e.d.k.s.s.PrintOutBolt - [execute,64] - Bolt的数据处理失败!数据:["2"]
java.util.NoSuchElementException: null
	at java.util.ArrayList$Itr.next(ArrayList.java:854)
	at com.example.dmy.kafkastorm.storm.bolt.PrintOutBolt.execute(PrintOutBolt.java:53)
	at org.apache.storm.daemon.executor$fn__10795$tuple_action_fn__10797.invoke(executor.clj:739)
	at org.apache.storm.daemon.executor$mk_task_receiver$fn__10716.invoke(executor.clj:468)
	at org.apache.storm.disruptor$clojure_handler$reify__10135.onEvent(disruptor.clj:41)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchToCursor(DisruptorQueue.java:509)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchWhenAvailable(DisruptorQueue.java:487)
	at org.apache.storm.disruptor$consume_batch_when_available.invoke(disruptor.clj:74)
	at org.apache.storm.daemon.executor$fn__10795$fn__10808$fn__10861.invoke(executor.clj:861)
	at org.apache.storm.util$async_loop$fn__553.invoke(util.clj:484)
	at clojure.lang.AFn.run(AFn.java:22)
	at java.lang.Thread.run(Thread.java:748)
16:02:50.806 [Thread-40-PrintOutBolt-executor[2 2]] ERROR c.e.d.k.s.s.PrintOutBolt - [execute,64] - Bolt的数据处理失败!数据:["0"]
java.util.NoSuchElementException: null
	at java.util.ArrayList$Itr.next(ArrayList.java:854)
	at com.example.dmy.kafkastorm.storm.bolt.PrintOutBolt.execute(PrintOutBolt.java:53)
	at org.apache.storm.daemon.executor$fn__10795$tuple_action_fn__10797.invoke(executor.clj:739)
	at org.apache.storm.daemon.executor$mk_task_receiver$fn__10716.invoke(executor.clj:468)
	at org.apache.storm.disruptor$clojure_handler$reify__10135.onEvent(disruptor.clj:41)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchToCursor(DisruptorQueue.java:509)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchWhenAvailable(DisruptorQueue.java:487)
	at org.apache.storm.disruptor$consume_batch_when_available.invoke(disruptor.clj:74)
	at org.apache.storm.daemon.executor$fn__10795$fn__10808$fn__10861.invoke(executor.clj:861)
	at org.apache.storm.util$async_loop$fn__553.invoke(util.clj:484)
	at clojure.lang.AFn.run(AFn.java:22)
	at java.lang.Thread.run(Thread.java:748)
16:02:52.810 [Thread-40-PrintOutBolt-executor[2 2]] ERROR c.e.d.k.s.s.PrintOutBolt - [execute,64] - Bolt的数据处理失败!数据:["5"]
java.util.NoSuchElementException: null
	at java.util.ArrayList$Itr.next(ArrayList.java:854)
	at com.example.dmy.kafkastorm.storm.bolt.PrintOutBolt.execute(PrintOutBolt.java:53)
	at org.apache.storm.daemon.executor$fn__10795$tuple_action_fn__10797.invoke(executor.clj:739)
	at org.apache.storm.daemon.executor$mk_task_receiver$fn__10716.invoke(executor.clj:468)
	at org.apache.storm.disruptor$clojure_handler$reify__10135.onEvent(disruptor.clj:41)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchToCursor(DisruptorQueue.java:509)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchWhenAvailable(DisruptorQueue.java:487)
	at org.apache.storm.disruptor$consume_batch_when_available.invoke(disruptor.clj:74)
	at org.apache.storm.daemon.executor$fn__10795$fn__10808$fn__10861.invoke(executor.clj:861)
	at org.apache.storm.util$async_loop$fn__553.invoke(util.clj:484)
	at clojure.lang.AFn.run(AFn.java:22)
	at java.lang.Thread.run(Thread.java:748)
16:02:54.811 [Thread-40-PrintOutBolt-executor[2 2]] ERROR c.e.d.k.s.s.PrintOutBolt - [execute,64] - Bolt的数据处理失败!数据:["4"]
java.util.NoSuchElementException: null
	at java.util.ArrayList$Itr.next(ArrayList.java:854)
	at com.example.dmy.kafkastorm.storm.bolt.PrintOutBolt.execute(PrintOutBolt.java:53)
	at org.apache.storm.daemon.executor$fn__10795$tuple_action_fn__10797.invoke(executor.clj:739)
	at org.apache.storm.daemon.executor$mk_task_receiver$fn__10716.invoke(executor.clj:468)
	at org.apache.storm.disruptor$clojure_handler$reify__10135.onEvent(disruptor.clj:41)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchToCursor(DisruptorQueue.java:509)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchWhenAvailable(DisruptorQueue.java:487)
	at org.apache.storm.disruptor$consume_batch_when_available.invoke(disruptor.clj:74)
	at org.apache.storm.daemon.executor$fn__10795$fn__10808$fn__10861.invoke(executor.clj:861)
	at org.apache.storm.util$async_loop$fn__553.invoke(util.clj:484)
	at clojure.lang.AFn.run(AFn.java:22)
	at java.lang.Thread.run(Thread.java:748)
16:07:01.227 [Thread-41-PrintOutBolt-executor[2 2]] ERROR c.e.d.k.s.s.PrintOutBolt - [execute,65] - Bolt的数据处理失败!数据:["8","6","1","1","7","1","2","9","7","2","8","1","5"]
java.util.NoSuchElementException: null
	at java.util.ArrayList$Itr.next(ArrayList.java:854)
	at com.example.dmy.kafkastorm.storm.bolt.PrintOutBolt.execute(PrintOutBolt.java:54)
	at org.apache.storm.daemon.executor$fn__10795$tuple_action_fn__10797.invoke(executor.clj:739)
	at org.apache.storm.daemon.executor$mk_task_receiver$fn__10716.invoke(executor.clj:468)
	at org.apache.storm.disruptor$clojure_handler$reify__10135.onEvent(disruptor.clj:41)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchToCursor(DisruptorQueue.java:509)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchWhenAvailable(DisruptorQueue.java:487)
	at org.apache.storm.disruptor$consume_batch_when_available.invoke(disruptor.clj:74)
	at org.apache.storm.daemon.executor$fn__10795$fn__10808$fn__10861.invoke(executor.clj:861)
	at org.apache.storm.util$async_loop$fn__553.invoke(util.clj:484)
	at clojure.lang.AFn.run(AFn.java:22)
	at java.lang.Thread.run(Thread.java:748)
16:07:03.221 [Thread-41-PrintOutBolt-executor[2 2]] ERROR c.e.d.k.s.s.PrintOutBolt - [execute,65] - Bolt的数据处理失败!数据:["7"]
java.util.NoSuchElementException: null
	at java.util.ArrayList$Itr.next(ArrayList.java:854)
	at com.example.dmy.kafkastorm.storm.bolt.PrintOutBolt.execute(PrintOutBolt.java:54)
	at org.apache.storm.daemon.executor$fn__10795$tuple_action_fn__10797.invoke(executor.clj:739)
	at org.apache.storm.daemon.executor$mk_task_receiver$fn__10716.invoke(executor.clj:468)
	at org.apache.storm.disruptor$clojure_handler$reify__10135.onEvent(disruptor.clj:41)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchToCursor(DisruptorQueue.java:509)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchWhenAvailable(DisruptorQueue.java:487)
	at org.apache.storm.disruptor$consume_batch_when_available.invoke(disruptor.clj:74)
	at org.apache.storm.daemon.executor$fn__10795$fn__10808$fn__10861.invoke(executor.clj:861)
	at org.apache.storm.util$async_loop$fn__553.invoke(util.clj:484)
	at clojure.lang.AFn.run(AFn.java:22)
	at java.lang.Thread.run(Thread.java:748)
16:07:05.245 [Thread-41-PrintOutBolt-executor[2 2]] ERROR c.e.d.k.s.s.PrintOutBolt - [execute,65] - Bolt的数据处理失败!数据:["8"]
java.util.NoSuchElementException: null
	at java.util.ArrayList$Itr.next(ArrayList.java:854)
	at com.example.dmy.kafkastorm.storm.bolt.PrintOutBolt.execute(PrintOutBolt.java:54)
	at org.apache.storm.daemon.executor$fn__10795$tuple_action_fn__10797.invoke(executor.clj:739)
	at org.apache.storm.daemon.executor$mk_task_receiver$fn__10716.invoke(executor.clj:468)
	at org.apache.storm.disruptor$clojure_handler$reify__10135.onEvent(disruptor.clj:41)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchToCursor(DisruptorQueue.java:509)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchWhenAvailable(DisruptorQueue.java:487)
	at org.apache.storm.disruptor$consume_batch_when_available.invoke(disruptor.clj:74)
	at org.apache.storm.daemon.executor$fn__10795$fn__10808$fn__10861.invoke(executor.clj:861)
	at org.apache.storm.util$async_loop$fn__553.invoke(util.clj:484)
	at clojure.lang.AFn.run(AFn.java:22)
	at java.lang.Thread.run(Thread.java:748)
16:07:07.270 [Thread-41-PrintOutBolt-executor[2 2]] ERROR c.e.d.k.s.s.PrintOutBolt - [execute,65] - Bolt的数据处理失败!数据:["5"]
java.util.NoSuchElementException: null
	at java.util.ArrayList$Itr.next(ArrayList.java:854)
	at com.example.dmy.kafkastorm.storm.bolt.PrintOutBolt.execute(PrintOutBolt.java:54)
	at org.apache.storm.daemon.executor$fn__10795$tuple_action_fn__10797.invoke(executor.clj:739)
	at org.apache.storm.daemon.executor$mk_task_receiver$fn__10716.invoke(executor.clj:468)
	at org.apache.storm.disruptor$clojure_handler$reify__10135.onEvent(disruptor.clj:41)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchToCursor(DisruptorQueue.java:509)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchWhenAvailable(DisruptorQueue.java:487)
	at org.apache.storm.disruptor$consume_batch_when_available.invoke(disruptor.clj:74)
	at org.apache.storm.daemon.executor$fn__10795$fn__10808$fn__10861.invoke(executor.clj:861)
	at org.apache.storm.util$async_loop$fn__553.invoke(util.clj:484)
	at clojure.lang.AFn.run(AFn.java:22)
	at java.lang.Thread.run(Thread.java:748)
16:07:09.295 [Thread-41-PrintOutBolt-executor[2 2]] ERROR c.e.d.k.s.s.PrintOutBolt - [execute,65] - Bolt的数据处理失败!数据:["1"]
java.util.NoSuchElementException: null
	at java.util.ArrayList$Itr.next(ArrayList.java:854)
	at com.example.dmy.kafkastorm.storm.bolt.PrintOutBolt.execute(PrintOutBolt.java:54)
	at org.apache.storm.daemon.executor$fn__10795$tuple_action_fn__10797.invoke(executor.clj:739)
	at org.apache.storm.daemon.executor$mk_task_receiver$fn__10716.invoke(executor.clj:468)
	at org.apache.storm.disruptor$clojure_handler$reify__10135.onEvent(disruptor.clj:41)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchToCursor(DisruptorQueue.java:509)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchWhenAvailable(DisruptorQueue.java:487)
	at org.apache.storm.disruptor$consume_batch_when_available.invoke(disruptor.clj:74)
	at org.apache.storm.daemon.executor$fn__10795$fn__10808$fn__10861.invoke(executor.clj:861)
	at org.apache.storm.util$async_loop$fn__553.invoke(util.clj:484)
	at clojure.lang.AFn.run(AFn.java:22)
	at java.lang.Thread.run(Thread.java:748)
16:07:11.317 [Thread-41-PrintOutBolt-executor[2 2]] ERROR c.e.d.k.s.s.PrintOutBolt - [execute,65] - Bolt的数据处理失败!数据:["2"]
java.util.NoSuchElementException: null
	at java.util.ArrayList$Itr.next(ArrayList.java:854)
	at com.example.dmy.kafkastorm.storm.bolt.PrintOutBolt.execute(PrintOutBolt.java:54)
	at org.apache.storm.daemon.executor$fn__10795$tuple_action_fn__10797.invoke(executor.clj:739)
	at org.apache.storm.daemon.executor$mk_task_receiver$fn__10716.invoke(executor.clj:468)
	at org.apache.storm.disruptor$clojure_handler$reify__10135.onEvent(disruptor.clj:41)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchToCursor(DisruptorQueue.java:509)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchWhenAvailable(DisruptorQueue.java:487)
	at org.apache.storm.disruptor$consume_batch_when_available.invoke(disruptor.clj:74)
	at org.apache.storm.daemon.executor$fn__10795$fn__10808$fn__10861.invoke(executor.clj:861)
	at org.apache.storm.util$async_loop$fn__553.invoke(util.clj:484)
	at clojure.lang.AFn.run(AFn.java:22)
	at java.lang.Thread.run(Thread.java:748)
16:07:13.341 [Thread-41-PrintOutBolt-executor[2 2]] ERROR c.e.d.k.s.s.PrintOutBolt - [execute,65] - Bolt的数据处理失败!数据:["8"]
java.util.NoSuchElementException: null
	at java.util.ArrayList$Itr.next(ArrayList.java:854)
	at com.example.dmy.kafkastorm.storm.bolt.PrintOutBolt.execute(PrintOutBolt.java:54)
	at org.apache.storm.daemon.executor$fn__10795$tuple_action_fn__10797.invoke(executor.clj:739)
	at org.apache.storm.daemon.executor$mk_task_receiver$fn__10716.invoke(executor.clj:468)
	at org.apache.storm.disruptor$clojure_handler$reify__10135.onEvent(disruptor.clj:41)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchToCursor(DisruptorQueue.java:509)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchWhenAvailable(DisruptorQueue.java:487)
	at org.apache.storm.disruptor$consume_batch_when_available.invoke(disruptor.clj:74)
	at org.apache.storm.daemon.executor$fn__10795$fn__10808$fn__10861.invoke(executor.clj:861)
	at org.apache.storm.util$async_loop$fn__553.invoke(util.clj:484)
	at clojure.lang.AFn.run(AFn.java:22)
	at java.lang.Thread.run(Thread.java:748)
16:07:15.363 [Thread-41-PrintOutBolt-executor[2 2]] ERROR c.e.d.k.s.s.PrintOutBolt - [execute,65] - Bolt的数据处理失败!数据:["7"]
java.util.NoSuchElementException: null
	at java.util.ArrayList$Itr.next(ArrayList.java:854)
	at com.example.dmy.kafkastorm.storm.bolt.PrintOutBolt.execute(PrintOutBolt.java:54)
	at org.apache.storm.daemon.executor$fn__10795$tuple_action_fn__10797.invoke(executor.clj:739)
	at org.apache.storm.daemon.executor$mk_task_receiver$fn__10716.invoke(executor.clj:468)
	at org.apache.storm.disruptor$clojure_handler$reify__10135.onEvent(disruptor.clj:41)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchToCursor(DisruptorQueue.java:509)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchWhenAvailable(DisruptorQueue.java:487)
	at org.apache.storm.disruptor$consume_batch_when_available.invoke(disruptor.clj:74)
	at org.apache.storm.daemon.executor$fn__10795$fn__10808$fn__10861.invoke(executor.clj:861)
	at org.apache.storm.util$async_loop$fn__553.invoke(util.clj:484)
	at clojure.lang.AFn.run(AFn.java:22)
	at java.lang.Thread.run(Thread.java:748)
16:07:17.385 [Thread-41-PrintOutBolt-executor[2 2]] ERROR c.e.d.k.s.s.PrintOutBolt - [execute,65] - Bolt的数据处理失败!数据:["7"]
java.util.NoSuchElementException: null
	at java.util.ArrayList$Itr.next(ArrayList.java:854)
	at com.example.dmy.kafkastorm.storm.bolt.PrintOutBolt.execute(PrintOutBolt.java:54)
	at org.apache.storm.daemon.executor$fn__10795$tuple_action_fn__10797.invoke(executor.clj:739)
	at org.apache.storm.daemon.executor$mk_task_receiver$fn__10716.invoke(executor.clj:468)
	at org.apache.storm.disruptor$clojure_handler$reify__10135.onEvent(disruptor.clj:41)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchToCursor(DisruptorQueue.java:509)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchWhenAvailable(DisruptorQueue.java:487)
	at org.apache.storm.disruptor$consume_batch_when_available.invoke(disruptor.clj:74)
	at org.apache.storm.daemon.executor$fn__10795$fn__10808$fn__10861.invoke(executor.clj:861)
	at org.apache.storm.util$async_loop$fn__553.invoke(util.clj:484)
	at clojure.lang.AFn.run(AFn.java:22)
	at java.lang.Thread.run(Thread.java:748)
16:07:19.407 [Thread-41-PrintOutBolt-executor[2 2]] ERROR c.e.d.k.s.s.PrintOutBolt - [execute,65] - Bolt的数据处理失败!数据:["8"]
java.util.NoSuchElementException: null
	at java.util.ArrayList$Itr.next(ArrayList.java:854)
	at com.example.dmy.kafkastorm.storm.bolt.PrintOutBolt.execute(PrintOutBolt.java:54)
	at org.apache.storm.daemon.executor$fn__10795$tuple_action_fn__10797.invoke(executor.clj:739)
	at org.apache.storm.daemon.executor$mk_task_receiver$fn__10716.invoke(executor.clj:468)
	at org.apache.storm.disruptor$clojure_handler$reify__10135.onEvent(disruptor.clj:41)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchToCursor(DisruptorQueue.java:509)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchWhenAvailable(DisruptorQueue.java:487)
	at org.apache.storm.disruptor$consume_batch_when_available.invoke(disruptor.clj:74)
	at org.apache.storm.daemon.executor$fn__10795$fn__10808$fn__10861.invoke(executor.clj:861)
	at org.apache.storm.util$async_loop$fn__553.invoke(util.clj:484)
	at clojure.lang.AFn.run(AFn.java:22)
	at java.lang.Thread.run(Thread.java:748)
16:07:21.410 [Thread-41-PrintOutBolt-executor[2 2]] ERROR c.e.d.k.s.s.PrintOutBolt - [execute,65] - Bolt的数据处理失败!数据:["0"]
java.util.NoSuchElementException: null
	at java.util.ArrayList$Itr.next(ArrayList.java:854)
	at com.example.dmy.kafkastorm.storm.bolt.PrintOutBolt.execute(PrintOutBolt.java:54)
	at org.apache.storm.daemon.executor$fn__10795$tuple_action_fn__10797.invoke(executor.clj:739)
	at org.apache.storm.daemon.executor$mk_task_receiver$fn__10716.invoke(executor.clj:468)
	at org.apache.storm.disruptor$clojure_handler$reify__10135.onEvent(disruptor.clj:41)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchToCursor(DisruptorQueue.java:509)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchWhenAvailable(DisruptorQueue.java:487)
	at org.apache.storm.disruptor$consume_batch_when_available.invoke(disruptor.clj:74)
	at org.apache.storm.daemon.executor$fn__10795$fn__10808$fn__10861.invoke(executor.clj:861)
	at org.apache.storm.util$async_loop$fn__553.invoke(util.clj:484)
	at clojure.lang.AFn.run(AFn.java:22)
	at java.lang.Thread.run(Thread.java:748)
16:07:23.431 [Thread-41-PrintOutBolt-executor[2 2]] ERROR c.e.d.k.s.s.PrintOutBolt - [execute,65] - Bolt的数据处理失败!数据:["9"]
java.util.NoSuchElementException: null
	at java.util.ArrayList$Itr.next(ArrayList.java:854)
	at com.example.dmy.kafkastorm.storm.bolt.PrintOutBolt.execute(PrintOutBolt.java:54)
	at org.apache.storm.daemon.executor$fn__10795$tuple_action_fn__10797.invoke(executor.clj:739)
	at org.apache.storm.daemon.executor$mk_task_receiver$fn__10716.invoke(executor.clj:468)
	at org.apache.storm.disruptor$clojure_handler$reify__10135.onEvent(disruptor.clj:41)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchToCursor(DisruptorQueue.java:509)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchWhenAvailable(DisruptorQueue.java:487)
	at org.apache.storm.disruptor$consume_batch_when_available.invoke(disruptor.clj:74)
	at org.apache.storm.daemon.executor$fn__10795$fn__10808$fn__10861.invoke(executor.clj:861)
	at org.apache.storm.util$async_loop$fn__553.invoke(util.clj:484)
	at clojure.lang.AFn.run(AFn.java:22)
	at java.lang.Thread.run(Thread.java:748)
16:07:25.434 [Thread-41-PrintOutBolt-executor[2 2]] ERROR c.e.d.k.s.s.PrintOutBolt - [execute,65] - Bolt的数据处理失败!数据:["0"]
java.util.NoSuchElementException: null
	at java.util.ArrayList$Itr.next(ArrayList.java:854)
	at com.example.dmy.kafkastorm.storm.bolt.PrintOutBolt.execute(PrintOutBolt.java:54)
	at org.apache.storm.daemon.executor$fn__10795$tuple_action_fn__10797.invoke(executor.clj:739)
	at org.apache.storm.daemon.executor$mk_task_receiver$fn__10716.invoke(executor.clj:468)
	at org.apache.storm.disruptor$clojure_handler$reify__10135.onEvent(disruptor.clj:41)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchToCursor(DisruptorQueue.java:509)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchWhenAvailable(DisruptorQueue.java:487)
	at org.apache.storm.disruptor$consume_batch_when_available.invoke(disruptor.clj:74)
	at org.apache.storm.daemon.executor$fn__10795$fn__10808$fn__10861.invoke(executor.clj:861)
	at org.apache.storm.util$async_loop$fn__553.invoke(util.clj:484)
	at clojure.lang.AFn.run(AFn.java:22)
	at java.lang.Thread.run(Thread.java:748)
16:07:27.456 [Thread-41-PrintOutBolt-executor[2 2]] ERROR c.e.d.k.s.s.PrintOutBolt - [execute,65] - Bolt的数据处理失败!数据:["3"]
java.util.NoSuchElementException: null
	at java.util.ArrayList$Itr.next(ArrayList.java:854)
	at com.example.dmy.kafkastorm.storm.bolt.PrintOutBolt.execute(PrintOutBolt.java:54)
	at org.apache.storm.daemon.executor$fn__10795$tuple_action_fn__10797.invoke(executor.clj:739)
	at org.apache.storm.daemon.executor$mk_task_receiver$fn__10716.invoke(executor.clj:468)
	at org.apache.storm.disruptor$clojure_handler$reify__10135.onEvent(disruptor.clj:41)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchToCursor(DisruptorQueue.java:509)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchWhenAvailable(DisruptorQueue.java:487)
	at org.apache.storm.disruptor$consume_batch_when_available.invoke(disruptor.clj:74)
	at org.apache.storm.daemon.executor$fn__10795$fn__10808$fn__10861.invoke(executor.clj:861)
	at org.apache.storm.util$async_loop$fn__553.invoke(util.clj:484)
	at clojure.lang.AFn.run(AFn.java:22)
	at java.lang.Thread.run(Thread.java:748)
16:07:29.462 [Thread-41-PrintOutBolt-executor[2 2]] ERROR c.e.d.k.s.s.PrintOutBolt - [execute,65] - Bolt的数据处理失败!数据:["8"]
java.util.NoSuchElementException: null
	at java.util.ArrayList$Itr.next(ArrayList.java:854)
	at com.example.dmy.kafkastorm.storm.bolt.PrintOutBolt.execute(PrintOutBolt.java:54)
	at org.apache.storm.daemon.executor$fn__10795$tuple_action_fn__10797.invoke(executor.clj:739)
	at org.apache.storm.daemon.executor$mk_task_receiver$fn__10716.invoke(executor.clj:468)
	at org.apache.storm.disruptor$clojure_handler$reify__10135.onEvent(disruptor.clj:41)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchToCursor(DisruptorQueue.java:509)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchWhenAvailable(DisruptorQueue.java:487)
	at org.apache.storm.disruptor$consume_batch_when_available.invoke(disruptor.clj:74)
	at org.apache.storm.daemon.executor$fn__10795$fn__10808$fn__10861.invoke(executor.clj:861)
	at org.apache.storm.util$async_loop$fn__553.invoke(util.clj:484)
	at clojure.lang.AFn.run(AFn.java:22)
	at java.lang.Thread.run(Thread.java:748)
16:11:01.990 [Thread-42-PrintOutBolt-executor[2 2]] ERROR c.e.d.k.s.s.PrintOutBolt - [execute,66] - Bolt的数据处理失败!数据:["1","9","7","5","7","0","1","4","9","4","2","3","1"]
java.util.NoSuchElementException: null
	at java.util.ArrayList$Itr.next(ArrayList.java:854)
	at com.example.dmy.kafkastorm.storm.bolt.PrintOutBolt.execute(PrintOutBolt.java:60)
	at org.apache.storm.daemon.executor$fn__10795$tuple_action_fn__10797.invoke(executor.clj:739)
	at org.apache.storm.daemon.executor$mk_task_receiver$fn__10716.invoke(executor.clj:468)
	at org.apache.storm.disruptor$clojure_handler$reify__10135.onEvent(disruptor.clj:41)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchToCursor(DisruptorQueue.java:509)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchWhenAvailable(DisruptorQueue.java:487)
	at org.apache.storm.disruptor$consume_batch_when_available.invoke(disruptor.clj:74)
	at org.apache.storm.daemon.executor$fn__10795$fn__10808$fn__10861.invoke(executor.clj:861)
	at org.apache.storm.util$async_loop$fn__553.invoke(util.clj:484)
	at clojure.lang.AFn.run(AFn.java:22)
	at java.lang.Thread.run(Thread.java:748)
16:11:03.960 [Thread-42-PrintOutBolt-executor[2 2]] ERROR c.e.d.k.s.s.PrintOutBolt - [execute,66] - Bolt的数据处理失败!数据:["2"]
java.util.NoSuchElementException: null
	at java.util.ArrayList$Itr.next(ArrayList.java:854)
	at com.example.dmy.kafkastorm.storm.bolt.PrintOutBolt.execute(PrintOutBolt.java:60)
	at org.apache.storm.daemon.executor$fn__10795$tuple_action_fn__10797.invoke(executor.clj:739)
	at org.apache.storm.daemon.executor$mk_task_receiver$fn__10716.invoke(executor.clj:468)
	at org.apache.storm.disruptor$clojure_handler$reify__10135.onEvent(disruptor.clj:41)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchToCursor(DisruptorQueue.java:509)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchWhenAvailable(DisruptorQueue.java:487)
	at org.apache.storm.disruptor$consume_batch_when_available.invoke(disruptor.clj:74)
	at org.apache.storm.daemon.executor$fn__10795$fn__10808$fn__10861.invoke(executor.clj:861)
	at org.apache.storm.util$async_loop$fn__553.invoke(util.clj:484)
	at clojure.lang.AFn.run(AFn.java:22)
	at java.lang.Thread.run(Thread.java:748)
16:11:05.963 [Thread-42-PrintOutBolt-executor[2 2]] ERROR c.e.d.k.s.s.PrintOutBolt - [execute,66] - Bolt的数据处理失败!数据:["2"]
java.util.NoSuchElementException: null
	at java.util.ArrayList$Itr.next(ArrayList.java:854)
	at com.example.dmy.kafkastorm.storm.bolt.PrintOutBolt.execute(PrintOutBolt.java:60)
	at org.apache.storm.daemon.executor$fn__10795$tuple_action_fn__10797.invoke(executor.clj:739)
	at org.apache.storm.daemon.executor$mk_task_receiver$fn__10716.invoke(executor.clj:468)
	at org.apache.storm.disruptor$clojure_handler$reify__10135.onEvent(disruptor.clj:41)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchToCursor(DisruptorQueue.java:509)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchWhenAvailable(DisruptorQueue.java:487)
	at org.apache.storm.disruptor$consume_batch_when_available.invoke(disruptor.clj:74)
	at org.apache.storm.daemon.executor$fn__10795$fn__10808$fn__10861.invoke(executor.clj:861)
	at org.apache.storm.util$async_loop$fn__553.invoke(util.clj:484)
	at clojure.lang.AFn.run(AFn.java:22)
	at java.lang.Thread.run(Thread.java:748)
16:11:07.964 [Thread-42-PrintOutBolt-executor[2 2]] ERROR c.e.d.k.s.s.PrintOutBolt - [execute,66] - Bolt的数据处理失败!数据:["5"]
java.util.NoSuchElementException: null
	at java.util.ArrayList$Itr.next(ArrayList.java:854)
	at com.example.dmy.kafkastorm.storm.bolt.PrintOutBolt.execute(PrintOutBolt.java:60)
	at org.apache.storm.daemon.executor$fn__10795$tuple_action_fn__10797.invoke(executor.clj:739)
	at org.apache.storm.daemon.executor$mk_task_receiver$fn__10716.invoke(executor.clj:468)
	at org.apache.storm.disruptor$clojure_handler$reify__10135.onEvent(disruptor.clj:41)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchToCursor(DisruptorQueue.java:509)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchWhenAvailable(DisruptorQueue.java:487)
	at org.apache.storm.disruptor$consume_batch_when_available.invoke(disruptor.clj:74)
	at org.apache.storm.daemon.executor$fn__10795$fn__10808$fn__10861.invoke(executor.clj:861)
	at org.apache.storm.util$async_loop$fn__553.invoke(util.clj:484)
	at clojure.lang.AFn.run(AFn.java:22)
	at java.lang.Thread.run(Thread.java:748)
16:11:09.969 [Thread-42-PrintOutBolt-executor[2 2]] ERROR c.e.d.k.s.s.PrintOutBolt - [execute,66] - Bolt的数据处理失败!数据:["1"]
java.util.NoSuchElementException: null
	at java.util.ArrayList$Itr.next(ArrayList.java:854)
	at com.example.dmy.kafkastorm.storm.bolt.PrintOutBolt.execute(PrintOutBolt.java:60)
	at org.apache.storm.daemon.executor$fn__10795$tuple_action_fn__10797.invoke(executor.clj:739)
	at org.apache.storm.daemon.executor$mk_task_receiver$fn__10716.invoke(executor.clj:468)
	at org.apache.storm.disruptor$clojure_handler$reify__10135.onEvent(disruptor.clj:41)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchToCursor(DisruptorQueue.java:509)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchWhenAvailable(DisruptorQueue.java:487)
	at org.apache.storm.disruptor$consume_batch_when_available.invoke(disruptor.clj:74)
	at org.apache.storm.daemon.executor$fn__10795$fn__10808$fn__10861.invoke(executor.clj:861)
	at org.apache.storm.util$async_loop$fn__553.invoke(util.clj:484)
	at clojure.lang.AFn.run(AFn.java:22)
	at java.lang.Thread.run(Thread.java:748)
16:11:11.973 [Thread-42-PrintOutBolt-executor[2 2]] ERROR c.e.d.k.s.s.PrintOutBolt - [execute,66] - Bolt的数据处理失败!数据:["0"]
java.util.NoSuchElementException: null
	at java.util.ArrayList$Itr.next(ArrayList.java:854)
	at com.example.dmy.kafkastorm.storm.bolt.PrintOutBolt.execute(PrintOutBolt.java:60)
	at org.apache.storm.daemon.executor$fn__10795$tuple_action_fn__10797.invoke(executor.clj:739)
	at org.apache.storm.daemon.executor$mk_task_receiver$fn__10716.invoke(executor.clj:468)
	at org.apache.storm.disruptor$clojure_handler$reify__10135.onEvent(disruptor.clj:41)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchToCursor(DisruptorQueue.java:509)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchWhenAvailable(DisruptorQueue.java:487)
	at org.apache.storm.disruptor$consume_batch_when_available.invoke(disruptor.clj:74)
	at org.apache.storm.daemon.executor$fn__10795$fn__10808$fn__10861.invoke(executor.clj:861)
	at org.apache.storm.util$async_loop$fn__553.invoke(util.clj:484)
	at clojure.lang.AFn.run(AFn.java:22)
	at java.lang.Thread.run(Thread.java:748)
16:11:13.978 [Thread-42-PrintOutBolt-executor[2 2]] ERROR c.e.d.k.s.s.PrintOutBolt - [execute,66] - Bolt的数据处理失败!数据:["0"]
java.util.NoSuchElementException: null
	at java.util.ArrayList$Itr.next(ArrayList.java:854)
	at com.example.dmy.kafkastorm.storm.bolt.PrintOutBolt.execute(PrintOutBolt.java:60)
	at org.apache.storm.daemon.executor$fn__10795$tuple_action_fn__10797.invoke(executor.clj:739)
	at org.apache.storm.daemon.executor$mk_task_receiver$fn__10716.invoke(executor.clj:468)
	at org.apache.storm.disruptor$clojure_handler$reify__10135.onEvent(disruptor.clj:41)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchToCursor(DisruptorQueue.java:509)
	at org.apache.storm.utils.DisruptorQueue.consumeBatchWhenAvailable(DisruptorQueue.java:487)
	at org.apache.storm.disruptor$consume_batch_when_available.invoke(disruptor.clj:74)
	at org.apache.storm.daemon.executor$fn__10795$fn__10808$fn__10861.invoke(executor.clj:861)
	at org.apache.storm.util$async_loop$fn__553.invoke(util.clj:484)
	at clojure.lang.AFn.run(AFn.java:22)
	at java.lang.Thread.run(Thread.java:748)
